Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd1','9606','Homo sapiens','Human','17122850','Redon_et_al_2006','Control Set',null,'We have constructed a first-generation CNV map of the human genome through the study of 270 individuals from four populations with ancestry in Europe, Africa or Asia (the HapMap collection). DNA from these individuals was screened for CNV using two complementary technologies: single-nucleotide polymorphism (SNP) genotyping arrays, and clone-based comparative genomic hybridization.','SNP genotyping analysis,Probe signal intensity,SNP genotyping analysis',null,'SNP array,BAC aCGH,SNP array','Affymetrix GeneChip Early Access Mapping 500K Set Array (250K_Nsp_SNP),Agilent,Affymetrix GeneChip Mapping 500K Early Access Array (250K_Sty_SNP)','GRCh37,GRCh37.p13,GRCh38,NCBI35','GCA_000001405.14,GCA_000001405.15,GCF_000001405.11,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd118','10090,10091,10092,10096,39442','Mus musculus,Mus musculus castaneus,Mus musculus domesticus,Mus musculus musculus,Mus spretus','Mouse','21921910','Keane_et_al_2011','Control Set','http://www.sanger.ac.uk/resources/mouse/genomes/','We report genome sequences of 17 inbred strains of laboratory mice and identify almost ten times more variants than previously known. We use these genomes to explore the phylogenetic history of the laboratory mouse and to examine the functional consequences of allele-specific variation on transcript abundance, revealing that at least 12% of transcripts show a significant tissue-specific expression bias. By identifying candidate functional variants at 718 quantitative trait loci we show that the molecular nature of functional variants and their position relative to genes vary according to the effect size of the locus. These sequences provide a starting point for a new era in the functional analysis of a key model organism.','Paired-end mapping,Read depth,Paired-end mapping,Paired-end mapping,Paired-end mapping,Split read mapping','Pindel,cnD,SECluster,BreakDancer,RetroSeq','Sequencing,Sequencing,Sequencing,Sequencing,Sequencing,Sequencing','Illumina,Illumina,Illumina,Illumina,Illumina,Illumina','MGSCv37','GCF_000001635.16');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd176','9606','Homo sapiens','Human','21479260','Banerjee_et_al_2011','Control Set',null,'We propose a three step computational framework (Identification of germline Changes in Copy Number or IgC2N) to discover and genotype germline CNVs. First, we detect candidate CNV loci by combining information across multiple samples without imposing restrictions to the number of coverage markers or to the variant size. Secondly, we fine tune the detection of rare variants and infer the putative copy number classes for each locus. Last, for each variant we combine the relative distance between consecutive copy number classes with genetic information in a novel attempt to estimate the reference model bias. This computational approach is applied to genome-wide data from 1250 HapMap individuals.','Other','Identification of germline Changes in Copy Number (IgC2N)','SNP array','Affymetrix SNP 6.0','GRCh37,GRCh37.p13,GRCh38,NCBI36','GCA_000001405.14,GCA_000001405.15,GCF_000001405.12,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd180','9606','Homo sapiens','Human','20482838','Pang_et_al_2010','Control Set',null,'Several genomes have now been sequenced, with millions of genetic variants annotated. While significant progress has been made in mapping single nucleotide polymorphisms (SNPs) and small (<10 bp) insertion/deletions (indels), the annotation of larger structural variants has been less comprehensive. It is still unclear to what extent a typical genome differs from the reference assembly, and the analysis of the genomes sequenced to date have shown varying results for copy number variation (CNV) and inversions. RESULTS: We have combined computational re-analysis of existing whole genome sequence data with novel microarray-based analysis, and detect 12,178 structural variants covering 40.6 Mb that were not reported in the initial sequencing of the first published personal genome. We estimate a total non-SNP variation content of 48.8 Mb in a single genome. Our results indicate that this genome differs from the consensus reference sequence by approximately 1.2% when considering indels/CNVs, 0.1% by SNPs and approximately 0.3% by inversions. The structural variants impact 4,867 genes, and >24% of structural variants would not be imputed by SNP-association. CONCLUSIONS: Our results indicate that a large number of structural variants have been unreported in the individual genomes published to date. This significant extent and complexity of structural variants, as well as the growing recognition of their medical relevance, necessitate they be actively studied in health-related analyses of personal genomes. The new catalogue of structural variants generated for this genome provides a crucial resource for future comparison studies.','Paired-end mapping,SNP genotyping analysis,Split read mapping,Probe signal intensity,SNP genotyping analysis,Probe signal intensity,Probe signal intensity',null,'Sequencing,SNP array,Sequencing,Oligo aCGH,SNP array,Oligo aCGH,Oligo aCGH','Sanger Sequencing,Not Submitted,Sanger Sequencing,Not Submitted,Not Submitted,Not Submitted,Not Submitted','GRCh37,GRCh37.p13,GRCh38,NCBI36','GCA_000001405.14,GCA_000001405.15,GCF_000001405.12,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd185','10090','Mus musculus','Mouse','22439878','Yalcin_et_al_2012','Control Set',null,'Accurate catalogues of structural variants (SVs) in mammalian genomes are necessary to give insights into potential mechanism of SV formation and to assess their functional impact. Next generation sequencing methods for SV detection are an advance on array based methods, but are almost exclusively limited to four basic categories: deletions, insertions, inversions and copy number gains, even though there has been an increasing number of complex genomic rearrangements associated with human pathologies. By visual inspection of over a 100 Mb of genome to which next generation sequence data from 17 inbred mouse strains had been aligned, followed by PCR and Sanger-based sequencing, we identify and interpret 21 paired-end mapping (PEM) patterns. These PEM patterns reveal a greater diversity and complexity in SVs than previously recognised. Analysis of breakpoint sequences at 261 SV sites revealed additional complexity at the nucleotide level of about a quarter of structural variants analysed. We found micro-deletions and micro-insertions at SV breakpoints ranging from 1 to 289 base pairs and SNPs that extended micro-homology associated with SV formation. An integrative approach using experimental analyses to train computational SV calling is essential for the accurate resolution of the architecture of SVs. We find considerable complexity in SV formation; about a quarter of SV in the mouse is made of a complex mixture of deletion, insertion, inversion and copy number gain. Computational methods can be adapted to identify most PEM patterns.','Manual observation,Manual observation,Paired-end mapping','Manual observation','Sequencing,PCR,Sequencing','Capillary (Sanger sequencing),PCR,Illumina GAIIx','MGSCv37','GCF_000001635.16');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd186','9606','Homo sapiens','Human','22693284','Thevenon_et_al_2012','Case-Set',null,'Identification of intragenic rearrangements affecting the CG-1 domain of the CAMTA1 gene, associated with autosomal dominant non-progressive congenital ataxia with or without intellectual diasbility','SNP genotyping analysis,Probe signal intensity,Probe signal intensity','Illumina cnvPartition 3.1.6','SNP array,Oligo aCGH,qPCR','Illumina HumanCytoSnp-12 Beadchip version 2.1,Agilent  105K CGH array,Roche LC-480','GRCh37,GRCh38,NCBI36','GCA_000001405.15,GCF_000001405.12,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd188','9606','Homo sapiens','Human','21552272','Pinto_et_al_2011','Control Set',null,'We have systematically compared copy number variant (CNV) detection on eleven microarrays to evaluate data quality and CNV calling, reproducibility, concordance across array platforms and laboratory sites, breakpoint accuracy and analysis tool variability. Different analytic tools applied to the same raw data typically yield CNV calls with &lt;50% concordance. Moreover, reproducibility in replicate experiments is &lt;70% for most platforms. Nevertheless, these findings should not preclude detection of large CNVs for clinical diagnostic purposes because large CNVs with poor reproducibility are found primarily in complex genomic regions and would typically be removed by standard clinical data curation. The striking differences between CNV calls from different platforms and analytic tools highlight the importance of careful assessment of experimental design in discovery and association studies and of strict data curation and filtering in diagnostics. The CNV resource presented here allows independent data evaluation and provides a means to benchmark new algorithms.','SNP genotyping analysis,SNP genotyping analysis,SNP genotyping analysis,SNP genotyping analysis,SNP genotyping analysis,SNP genotyping analysis,Probe signal intensity,SNP genotyping analysis,SNP genotyping analysis,SNP genotyping analysis,SNP genotyping analysis,Probe signal intensity,Probe signal intensity,Probe signal intensity,SNP genotyping analysis,SNP genotyping analysis,SNP genotyping analysis,SNP genotyping analysis,SNP genotyping analysis,SNP genotyping analysis,SNP genotyping analysis,SNP genotyping analysis,SNP genotyping analysis,Probe signal intensity,SNP genotyping analysis,SNP genotyping analysis,SNP genotyping analysis,SNP genotyping analysis,SNP genotyping analysis,SNP genotyping analysis,SNP genotyping analysis,SNP genotyping analysis,SNP genotyping analysis,SNP genotyping analysis,SNP genotyping analysis,SNP genotyping analysis,Probe signal intensity,Probe signal intensity,SNP genotyping analysis,SNP genotyping analysis,Probe signal intensity,Probe signal intensity,SNP genotyping analysis,SNP genotyping analysis,SNP genotyping analysis,SNP genotyping analysis,SNP genotyping analysis,SNP genotyping analysis,Probe signal intensity','PCNV,PCNV,cnvPart,cnvPart,Partek,QSNP,ADM2,QSNP,Partek,cnvPart,QSNP,Nexus,ADM2,Nexus,GTC-CN4,dChip,GTC-CN5,dChip,PCNV,iPattern,cnvPart,cnvPart,PCNV,cnvFinder,QSNP,cnvPart,QSNP,Birdsuite,iPattern,PCNV,dChip,Nexus,QSNP,PCNV,iPattern,iPattern,Nexus,ADM2,cnvPart,iPattern,Nexus,Nexus,PCNV,QSNP,GTC-CN5,iPattern,iPattern,Birdsuite,Nexus','SNP array,SNP array,SNP array,SNP array,SNP array,SNP array,Oligo aCGH,SNP array,SNP array,SNP array,SNP array,Oligo aCGH,Oligo aCGH,Oligo aCGH,SNP array,SNP array,SNP array,SNP array,SNP array,SNP array,SNP array,SNP array,SNP array,BAC aCGH,SNP array,SNP array,SNP array,SNP array,SNP array,SNP array,SNP array,SNP array,SNP array,SNP array,SNP array,SNP array,Oligo aCGH,Oligo aCGH,SNP array,SNP array,BAC aCGH,Oligo aCGH,SNP array,SNP array,SNP array,SNP array,SNP array,SNP array,Oligo aCGH','Not specified,Not specified,Not specified,Not specified,Not specified,Not specified,Not specified,Not specified,Not specified,Not specified,Not specified,Not specified,Not specified,Not specified,Not specified,Not specified,Not specified,Not specified,Not specified,Not specified,Not specified,Not specified,Not specified,Not specified,Not specified,Not specified,Not specified,Not specified,Not specified,Not specified,Not specified,Not specified,Not specified,Not specified,Not specified,Not specified,Not specified,Not specified,Not specified,Not specified,Not specified,Not specified,Not specified,Not specified,Not specified,Not specified,Not specified,
Not specified,Not specified','GRCh37,GRCh37.p13,GRCh38,NCBI36','GCA_000001405.14,GCA_000001405.15,GCF_000001405.12,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd19','9606','Homo sapiens','Human','19470904','Ahn_et_al_2009','Control Set',null,'We sequenced the genome of one Korean individual to 28.95-fold redundancy using Illumina paired-end sequencing. We identified 4298 structural variants.','Read depth and paired-end mapping',null,'Sequencing','Illumina Genome Analyzer','GRCh37,GRCh37.p13,GRCh38,NCBI36','GCA_000001405.14,GCA_000001405.15,GCF_000001405.12,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd192','9606','Homo sapiens','Human','18428421,20952405','COSMIC','Collection','http://cancer.sanger.ac.uk/cancergenome/projects/cosmic/','Catalogue of Somatic Mutations in Cancer (COSMIC) version 71 - All cancers arise as a result of the acquisition of a series of fixed DNA sequence abnormalities, mutations, many of which ultimately confer a growth advantage upon the cells in which they have occurred. There is a vast amount of information available in the published scientific literature about these changes. COSMIC is designed to store and display somatic mutation information and related details and contains information relating to human cancers.','Curated',null,'Curated','Curated','GRCh37','GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd193','9606','Homo sapiens','Human','16254605','Feuk_et_al_2005','Control Set',null,'With a draft genome-sequence assembly for the chimpanzee available, it is now possible to perform genome-wideanalyses to identify, at a submicroscopic level, structural rearrangements that have occurred between chimpanzeesand humans. The goal of this study was to investigate chromosomal regions that are inverted between the chimpanzeeand human genomes. Using the net alignments for the builds of the human and chimpanzee genome assemblies, weidentified a total of 1,576 putative regions of inverted orientation, covering more than 154 mega-bases of DNA. TheDNA segments are distributed throughout the genome and range from 23 base pairs to 62 mega-bases in length. Forthe 66 inversions more than 25 kilobases (kb) in length, 75% were flanked on one or both sides by (often unrelated)segmental duplications. Using PCR and fluorescence in situ hybridization we experimentally validated 23 of 27 (85%)semi-randomly chosen regions; the largest novel inversion confirmed was 4.3 mega-bases at human Chromosome7p14. Gorilla was used as an out-group to assign ancestral status to the variants. All experimentally validated inversionregions were then assayed against a panel of human samples and three of the 23 (13%) regions were found to bepolymorphic in the human genome. These polymorphic inversions include 730 kb (at 7p22), 13 kb (at 7q11), and 1 kb(at 16q24) fragments with a 5%, 30%, and 48% minor allele frequency, respectively. Our results suggest that inversionsare an important source of variation in primate genome evolution. The finding of at least three novel inversionpolymorphisms in humans indicates this type of structural variation may be a more common feature of our genomethan previously realized.','Manual observation,Manual observation',null,'FISH,PCR','Fluorescent In-Situ Hybridization Instruments,Applied Biosystem GeneAmp 9700','GRCh37,GRCh38,NCBI35','GCA_000001405.15,GCF_000001405.11,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd194','9606','Homo sapiens','Human','18987734','Bentley_et_al_2008','Control Set',null,'DNA sequence information underpins genetic research, enabling discoveries of important biological or medical benefit. Sequencing projects have traditionally used long (400-800 base pair) reads, but the existence of reference sequences for the human and many other genomes makes it possible to develop new, fast approaches to re-sequencing, whereby shorter reads are compared to a reference to identify intraspecies genetic variation. Here we report an approach that generates several billion bases of accurate nucleotide sequence per experiment at low cost. Single molecules of DNA are attached to a flat surface, amplified in situ and used as templates for synthetic sequencing with fluorescent reversible terminator deoxyribonucleotides. Images of the surface are analysed to generate high-quality sequence.We demonstrate application of this approach to human genome sequencing on flow-sorted X chromosomes and then scale the approach to determine the genome sequence of a male Yoruba from Ibadan, Nigeria. We build an accurate consensus sequence from.303 average depth of paired 35-base reads. We characterize four million single-nucleotide polymorphisms and four hundred thousand structural variants, many of which were previously unknown. Our approach is effective for accurate, rapid and economical whole-genome re-sequencing and many other biomedical applications.','Paired-end mapping,Split read mapping','ELAND SV Calling,MAQ InDel Calling','Sequencing,Sequencing','Not specified,Not specified','GRCh37,GRCh37.p13,GRCh38,NCBI36','GCA_000001405.14,GCA_000001405.15,GCF_000001405.12,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd195','9606','Homo sapiens','Human','20811451','Altshuler_et_al_2010','Control Set',null,'Despite great progress in identifying genetic variants that influence human disease,most inherited risk remains unexplained. A more complete understanding requires genome-wide studies that fully examine less common alleles in populations with a wide range of ancestry. To inform the design and interpretation of such studies, we genotyped 1.6 million common single nucleotide polymorphisms (SNPs) in 1,184 reference individuals from 11 global populations, and sequenced ten 100-kilobase regions in 692 of these individuals. This integrated data set of common and rare alleles, called ''HapMap 3'', includes both SNPs and copy number polymorphisms (CNPs). We characterized population-specific differences among low-frequency variants, measured the improvement in imputation accuracy afforded by the larger reference panel, especially in imputing SNPs with a minor allele frequency of less than or equal to 5%, and demonstrated the feasibility of imputing newly discovered CNPs and SNPs. This expanded public resource of genome variants in global populations supports deeper interrogation of genomic variation and its role in human disease, and serves as a step towards a high-resolution map of the landscape of human genetic variation.','SNP genotyping analysis','QuantiSNP, Birdseye','SNP array','Not specified','GRCh37,GRCh37.p13,GRCh38,NCBI36','GCA_000001405.14,GCA_000001405.15,GCF_000001405.12,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd196','9606','Homo sapiens','Human','17116639','Simon-Sanchez_et_al_2007','Control Set',null,'The recent hapmap effort has placed focus on the application of genome-wide SNP analysis to assess the contribution of genetic variability, particularly SNPs, to traits such as disease. Here, we describe the utility of genome-wide SNP analysis in the direct detection of extended homozygosity and structural genomic variation. We use this approach to assess the frequency of genomic alterations resulting from the lymphoblast immortalization and culture processes commonly used in cell repositories. We have assayed 408 804 SNPs in 276 DNA samples extracted from Epstein-Barr virus immortalized cell lines, which were derived from lymphocytes of elderly neurologically normal subjects. These data reveal extended homozygosity (contiguous tracts >5 Mb) in 9.5% (26/272) and 340 structural genomic alterations in 182 (66.9%) DNA samples assessed, 66% of which did not overlap with previously described structural variations. Examination of DNA extracted directly from the blood of 30 of these subjects confirmed all examined instances of extended homozygosity (6/6), 75% of structural genomic alteration <5 Mb in size (12/16) and 13% (1/8) of structural genomic alteration >5 Mb in size. These data suggest that structural genomic variation is a common phenomenon in the general population. While a proportion of this variability may be caused or its relative abundance altered by the immortalization and clonal process this will have only a minor effect on genotype and allele frequencies in a large cohort. It is likely that this powerful methodology will augment existing techniques in the identification of chromosomal abnormalities.','SNP genotyping analysis,Other','GenomeViewer tool within BeadStudio v2.2.22','SNP array,qPCR','Not specified,Applied Biosystems TaqMan','GRCh37,GRCh37.p13,GRCh38,NCBI35','GCA_000001405.14,GCA_000001405.15,GCF_000001405.11,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd197','9606','Homo sapiens','Human','19546169','McKernan_et_al_2009','Control Set',null,'We describe the genome sequencing of an anonymous individual of African origin using a novel ligation-based sequencing assay that enables a unique form of error correction that improves the raw accuracy of the aligned reads to greater than 99.9%, allowing us to accurately call SNPs with as few as two reads per allele. We collected several billion mate-paired reads yielding 18x haploid coverage of aligned sequence and close to 300x clone coverage. Over 98% of the reference genome is covered with at least one uniquely placed read, and 99.65% is spanned by at least one uniquely placed matepaired clone. We identify over 3.8 million SNPs, 19% of which are novel. Mate-paired data are used to physically resolve haplotype phases of nearly two-thirds of the genotypes obtained and produce phased segments of up to 215 kb. We detect 226,529 intra-read indels, 5590 indels between mate-paired reads, 91 inversions, and four gene fusions. We use a novel approach for detecting indels between mate-paired reads that are smaller than the standard deviation of the insert size of the library and discover deletions in common with those detected with our intra-read approach. Dozens of mutations previously described in OMIM and hundreds of nonsynonymous single-nucleotide and structural variants in genes previously implicated in disease are identified in this individual. There is more genetic variation in the human genome still to be uncovered, and we provide guidance for future surveys in populations and cancer biopsies.','Paired-end mapping,Paired-end mapping,Read depth,Split read mapping','AB Inversion Tool,AB_SOLiD Large Indel Tool,SOLiD Human CNV Tool,AB Small indel tool','Sequencing,Sequencing,Sequencing,Sequencing','Not specified,Not specified,Not specified,Not specified','GRCh37,GRCh37.p13,GRCh38,NCBI36','GCA_000001405.14,GCA_000001405.15,GCF_000001405.12,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd198','9606','Homo sapiens','Human','23635498','Chia_et_al_2012','Control Set',null,'We performed high density array analysis of 64 unrelated healthy Caucasian females using Illumina Omni1-Quad SNP microarray. Two CNV detection algorithms were applied to the raw data and the calls were merged to ensure a robust analysis of copy number variation. The results of a comprehensive investigation of CNVs on chromosome 18 in relation to distribution, incidence, gene density, population frequency and association with genomic architecture are described. Two CNV regions were confirmed by sequencing and precise breakpoints defined demonstrating a 2bp microhomology at the breakpoint junction of both CNVs. The investigation of associations of CNVs with repetitive elements on chromosome 18 shows that one third do not involve breakpoints in repeat sequences and the results of investigations suggests that the mechanism of derivation of CNVs in the normal population may be multi-factorial. The population frequencies of two CNVs were determined using a simple screening test that was designed to differentiate genotypes. This is the first specific investigation of CNVs on chromosome 18 with high resolution microarray of normal individuals using DNA extracted from whole blood.','Manual observation,Sequence alignment,SNP genotyping analysis,Merging,Manual observation,Manual observation,SNP genotyping analysis','PennCNV,CNV Partition','Karyotyping,Sequencing,SNP array,Merging,PCR,PCR,SNP array','Not specified,Not specified,IlluminaOmni1-QUAD,Merging,Not specified,Not specified,IlluminaOmni1-QUAD','GRCh37,GRCh37.p13,GRCh38,NCBI36','GCA_000001405.14,GCA_000001405.15,GCF_000001405.12,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd199','9606','Homo sapiens','Human','23128226','1000_Genomes_Consortium_Phase_1','Control Set','http://www.1000genomes.org','Through characterising the geographic and functional spectrum of human genetic variation, the 1000 Genomes Project aims to build a resource to help understand the genetic contribution to disease. Following from the Pilot Phase, in which we established the key principles underlying the project design, we now report on the genomes of 1,092 individuals drawn from 14 populations, constructed using a combination of low-coverage whole- genome and exome targeted sequencing. By developing methodologies to combine information across multiple algorithms and diverse data sources we provide an integrated and validated haplotype map of 38 million SNPs, 1.4 million indels and over 14 thousand larger deletions. We show how individuals from different populations have different profiles of rare and common variants and that low-frequency variants show elevated geographic differentiation, which is further increased by the action of purifying selection.
By measuring the excess of rare alleles, we show that evolutionary conservation and coding consequence are key determinants of the strength of purifying selection, and that rare- variant load varies substantially across biological pathways. We show that each individual harbours hundreds of rare, non-coding variants, such as transcription-factor-motif disrupting changes at conserved sites. This resource, which captures up to 98% of variants at 1% frequency in populations of medical genetics focus, enables imputation of common and low-frequency variants in individuals from diverse, including admixed, populations.','Read depth and paired-end mapping,Probe signal intensity,Genotyping,Split read mapping,Paired-end mapping,Other,Merging,Probe signal intensity,Paired-end mapping,Read depth,Genotyping','Genome STRiP,Genome STRiP, Beagle, Thunder,Pindel,DELLY,BreakDancerMax1.1,CNVnator 0.2.2,Beagle, Thunder','Sequencing,SNP array,Sequencing,Sequencing,Sequencing,PCR,Merging,Oligo aCGH,Sequencing,Sequencing,Sequencing','Illumina Genome Analyzer,Genome Analyzer II,Genome Analyzer IIx,HiSeq 2000,Illumina Omni 2.5,Illumina Genome Analyzer,Genome Analyzer II,Genome Analyzer IIx,HiSeq 2000,Illumina Genome Analyzer,Genome Analyzer II,Genome Analyzer IIx,HiSeq 2000,Illumina Genome Analyzer,Genome Analyzer II,Genome Analyzer IIx,HiSeq 2000,PTC-225 DNA Engine Tetrad Cycler (Biorad),Merging,Agilent 2x1M CGH Microarray,Illumina Genome Analyzer,Genome Analyzer II,Genome Analyzer IIx,HiSeq 2000,Illumina Genome Analyzer,Genome Analyzer II,Genome Analyzer IIx,HiSeq 2000,Illumina Genome Analyzer,Genome Analyzer II,Genome Analyzer IIx,HiSeq 2000,Roche 454 GS-FLX+,AB SOLiD','GRCh37,GRCh38','GCA_000001405.15,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd20','9606','Homo sapiens','Human','19812545','Conrad_et_al_2009','Control Set',null,'Structural variations of DNA greater than 1 kilobase in size account for most bases that vary among human genomes, but are still relatively under-ascertained. Here we use tiling oligonucleotide microarrays, comprising 42 million probes, to generate a comprehensive map of 11,700 copy number variations (CNVs) greater than 443 base pairs, of which most (8,599) have been validated independently. For 4,978 of these CNVs, we generated reference genotypes from 450 individuals of European, African or East Asian ancestry. The predominant mutational mechanisms differ among CNV size classes. Retrotransposition has duplicated and inserted some coding and non-coding DNA segments randomly around the genome. Furthermore, by correlation with known trait-associated single nucleotide polymorphisms (SNPs), we identified 30 loci with CNVs that are candidates for influencing disease susceptibility. Despite this, having assessed the completeness of our map and the patterns of linkage disequilibrium between CNVs and SNPs, we conclude that, for complex traits, the heritability void left by genome-wide association studies will not be accounted for by common CNVs.','Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity','GADA,GADA,GADA,GADA,GADA,GADA,GADA,GADA,GADA,GADA,GADA,GADA,GADA,GADA,GADA,GADA,GADA,GADA,GADA,GADA,GADA,GADA,GADA,GADA,GADA,GADA,GADA,GADA,GADA,GADA,GADA,GADA,GADA,GADA,GADA,GADA,GADA,GADA,GADA,GADA,GADA,GADA,GADA,GADA,GADA,GADA,GADA,GADA,GADA,GADA,GADA,GADA,GADA,GADA,GADA,GADA,GADA,GADA,GADA,GADA','Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH','Sanger H. Sapiens 42mCGH Array 5781_53 726K v1,Sanger H. Sapiens custom Agilent 105kCGH CNV Genotyping Array AMADID021270,Sanger H. Sapiens 42mCGH Array 5748_21 726K v1,Sanger H. Sapiens 42mCGH Array 
5739_11 726K v1,Sanger H. Sapiens 42mCGH Array 5754_27 726K v1,Sanger H. Sapiens 42mCGH Array 5754_26 726K v1,Sanger H. Sapiens 42mCGH Array 5745_17 726K v1,Sanger H. Sapiens 42mCGH Array 5763_35 726K v1,Sanger H. Sapiens 42mCGH Array 5781_52 726K v1,Sanger H. Sapiens 42mCGH Array 5739_10 726K v1,Sanger H. Sapiens 42mCGH Array 5733_6 726K v1,Sanger H. Sapiens 42mCGH Array 5757_28 726K v1,Sanger H. Sapiens 42mCGH Array 5775_48 726K v1,Sanger H. Sapiens 42mCGH Array 5775_47 726K v1,Sanger H. Sapiens 42mCGH Array 5769_42 726K v1,Sanger H. Sapiens 42mCGH Array 5748_19 726K v1,Sanger H. Sapiens 42mCGH Array 5784_56 726K v1,Sanger H. Sapiens 42mCGH Array 5748_20 726K v1,Sanger H. Sapiens 42mCGH Array 5730_2 726K v1,Sanger H. Sapiens 42mCGH Array 5739_12 726K v1,Sanger H. Sapiens 42mCGH Array 5742_13 726K v1,Sanger H. Sapiens 42mCGH Array 5730_1 726K v1,Sanger H. Sapiens 42mCGH Array 5772_44 726K v1,Sanger H. Sapiens 42mCGH Array 5784_55 726K v1,Sanger H. Sapiens 42mCGH Array 5745_16 726K v1,Sanger H. Sapiens 42mCGH Array 5772_45 726K v1,Sanger H. Sapiens 42mCGH Array 5736_7 726K v1,Sanger H. Sapiens 42mCGH Array 5751_22 726K v1,Sanger H. Sapiens 42mCGH Array 5787_60 726K v1,Sanger H. Sapiens 42mCGH Array 5778_49 726K v1,Sanger H. Sapiens 42mCGH Array 5781_54 726K v1,Sanger H. Sapiens 42mCGH Array 5760_31 726K v1,Sanger H. Sapiens 42mCGH Array 5736_9 726K v1,Sanger H. Sapiens 42mCGH Array 5742_15 726K v1,Sanger H. Sapiens 42mCGH Array 5760_32 726K v1,Sanger H. Sapiens 42mCGH Array 5778_50 726K v1,Sanger H. Sapiens 42mCGH Array 5766_38 726K v1,Sanger H. Sapiens 42mCGH Array 5751_23 726K v1,Sanger H. Sapiens 42mCGH Array 5787_58 726K v1,Sanger H. Sapiens 42mCGH Array 5769_40 726K v1,Sanger H. Sapiens 42mCGH Array 5778_51 726K v1,Sanger H. Sapiens 42mCGH Array 5733_4 726K v1,Sanger H. Sapiens 42mCGH Array 5760_33 726K v1,Sanger H. Sapiens 42mCGH Array 5757_29 726K v1,Sanger H. Sapiens 42mCGH Array 5787_59 726K v1,Sanger H. Sapiens 42mCGH Array 5775_46 726K v1,Sanger H. Sapiens 42mCGH Array 5754_25 726K v1,Sanger H. Sapiens 42mCGH Array 5772_43 726K v1,Sanger H. Sapiens 42mCGH Array 5745_18 726K v1,Sanger H. Sapiens 42mCGH Array 5784_57 726K v1,Sanger H. Sapiens 42mCGH Array 5763_34 726K v1,Sanger H. Sapiens 42mCGH Array 5751_24 726K v1,Sanger H. Sapiens 42mCGH Array 5736_8 726K v1,Sanger H. Sapiens 42mCGH Array 5763_36 726K v1,Sanger H. Sapiens 42mCGH Array 5769_41 726K v1,Sanger H. Sapiens 42mCGH Array 5766_37 726K v1,Sanger H. Sapiens 42mCGH Array 5757_30 726K v1,Sanger H. Sapiens 42mCGH Array 5730_3 726K v1,Sanger H. Sapiens 42mCGH Array 5742_14 726K v1,Sanger H. Sapiens 42mCGH Array 5733_5 726K v1,Sanger H. Sapiens 42mCGH Array 5766_39 726K v1','GRCh37,GRCh37.p13,GRCh38,NCBI36','GCA_000001405.14,GCA_000001405.15,GCF_000001405.12,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd200','10090','Mus musculus','Mouse','22916792','Wong_et_al_2012','Control Set',null,'The FVB/NJ mouse strain has its origins in a colony of outbred Swiss mice established in 1935 at the National Institutes of Health (NIH). Subsequent selective breeding for sensitivity to histamine diphosphate and the B strain of Friend leukemia virus led to the establishment of the FVB/N inbred strain, which was imported to the Jackson Laboratory and designated FVB/NJ. The FVB/NJ mouse has several distinct characteristics, such as large pronuclear morphology, vigorous reproductive performance, and consistently large litters that make it highly desirable for transgenic strain production and general purpose use. Using next-generation sequencing technology, we have sequenced the genome of FVB/NJ to approximately 50-fold coverage, and have generated a comprehensive catalogue of single nucleotide polymorphisms (SNPs), small insertion/deletions (indels), and structural variants (SVs), relative to the reference C57BL/6J genome. The sequencing and generation of this catalogue will help accelerate the identification of the precise molecular variants that are responsible for phenotypes observed in the FVB/NJ strain. We have examined a previously identified quantitative trait locus for atherosclerosis susceptibility on chromosome 10 and identify several previously unknown candidate causal variants.','Merging,Paired-end mapping,Merging,Merging,Merging,Merging,Paired-end mapping,Merging,Merging,Paired-end mapping,Local sequence assembly,Merging,Read depth,Merging,Paired-end mapping','PatternCheck,BreakDancer,SECluster,RetroSeq,SVmerge,cnD','Merging,Sequencing,Merging,Merging,Merging,Merging,Sequencing,Merging,Merging,Sequencing,Sequencing,Merging,Sequencing,Merging,Sequencing','Merging,Illumina HiSeq,Merging,Merging,Merging,Merging,Illumina HiSeq,Merging,Merging,Illumina HiSeq,Illumina HiSeq,Merging,Illumina HiSeq,Merging,Illumina HiSeq','MGSCv37','GCF_000001635.16');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd201','9606','Homo sapiens','Human','23290073','Wong_et_al_2012b','Control Set',null,'Whole-genome sequencing across multiple samples in a population provides an unprecedented opportunity to comprehensively characterize the polymorphic variants in the population. While the 1000 Genomes Project (1KGP) has offered brief insights into the value of population-level sequencing, the low coverage inadvertently compromised the ability to confidently detect rare and low-frequency variants. In addition, the composition of populations in the 1KGP is not complete, despite the extension of the study design to more than 2,500 samples from more than 20 population groups. The Malays are one of the Austronesian groups predominantly present in Southeast Asia and Oceania, and the Singapore Sequencing Malay Project (SSM) aims to perform deep whole-genome sequencing of 100 healthy Malays. Sequencing at an average of 30-fold coverage, we illustrate the higher sensitivity at detecting low-frequency and rare variants, and the ability to investigate the presence of hotspots of functional mutations. The deeper coverage allows more functional variants to be identified for each person when compared to the low-pass sequencing in 1KGP. This set of whole-genome sequence data is expected to be the benchmark for evaluating the value of deep population-level sequencing versus low-pass sequencing, especially in populations that are poorly represented in population genetic studies. We also expect the high coverage will enable methodological and technological assessments of current strategies in sequence data analysis.','Paired-end mapping','Breakdancer, VariationHunter','Sequencing','Illumina HiSeq 2000','GRCh37,GRCh38','GCA_000001405.15,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd203','9606','Homo sapiens','Human','21179565','Vogler_et_al_2010','Control Set',null,'In this study a cross-population microarray-based map of copy-number variant regions (CNVRs) was generated. We used the Affymetrix Genome-Wide Human SNP Array 6.0 to scan the genomes of 1167 individuals from two ethnically distinct populations (Europe, N = 717; Rwanda, N = 450). Three different CNV-finding algorithms were tested and compared for sensitivity, specificity, and feasibility. Two algorithms were subsequently used to construct CNVR maps, which were also validated by processing subsamples with additional microarray platforms (Illumina 1M-Duo BeadChip, Nimblegen 385K aCGH array) and by comparing our data with publicly available information.','SNP genotyping analysis,Merging,SNP genotyping analysis','PennCNV,Birdseye','SNP array,Merging,SNP array','Affymetrix 6.0 Human SNP Array,Merging,Affymetrix 6.0 Human SNP Array','GRCh37,GRCh37.p13,GRCh38,NCBI36','GCA_000001405.14,GCA_000001405.15,GCF_000001405.12,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd204','10090','Mus musculus','Mouse','23902802','Simon_et_al_2013','Control Set',null,'A study of phenotypic and genomic differences between C57BL/6J and C57BL/6N inbred mouse strains','Paired-end mapping,Manual observation',null,'Sequencing,PCR','Illumina,PCR','MGSCv37','GCF_000001635.16');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd205','7227','Drosophila melanogaster','Fruit fly','23222910','Zichner_et_al_2012','Control Set',null,'Genomic structural variation (SV) is a major determinant for phenotypic variation. Here, we report a highly accurate, densely validated map of unbalanced SVs comprising 8962 deletions and 916 tandem duplications in 39 Drosophila lines derived from a natural population (the "Drosophila melanogaster Genetic Reference Panel," DGRP). Most SVs (>90%) were inferred at nucleotide resolution, and a large fraction was genotyped across all samples. Based on this set of variants, we analyzed the formation mechanisms, genomic hotspots, and the phenotypic effect of SVs in Drosophila. Furthermore, we investigated SVs of three laboratory strain samples.','Merging,Read depth,Read depth and paired-end mapping,Split read mapping,Split read mapping,Read depth and paired-end mapping,Merging,Merging,Merging,Split read and paired-end mapping,Split read and paired-end mapping,Read depth','To combine inferred SVs from the different methods, we first merged the predictions of all 3 laboratory strain samples for each method individually. To do so, we defined confidence intervals around the breakpoints, according to the presumed resolution of breakpoint inference for each method (for SVs at nucleotide resolution: +/-10 bp, accounting for misalignments owing to short (<10 bp) indels; for DELLY: 50 bp outwards, 250 bp inwards; for CNVnator: 1 kb outwards, 400 bp inwards). Calls with overlapping confidence intervals at both SV start and end coordinates were combined into a single event. (For Genome STRiP, a method that performs population-inference of SVs in several samples at once, this merging step was not necessary.) Second, we filtered the individual lists. For DELLY, we required that on average at least 20 read pairs with a mapping quality of at least 20 supported the event in the individual samples. For Genome STRiP, we only considered SVs inferred with more than 20 supporting read pairs passing the internal filtering. For Pindel as well as CNVnator we did not apply any further filtering. As third step, we defined confidence intervals for each of the filtered lists (intervals as above; for Genome STRiP: 50 bp outwards, 250 bp inwards), and merged the entire SV set based on these confidence intervals. To increase the positive predictive value we filtered out SVs which were predicted by a single method only, with the exception that we kept 48 deletion calls solely made by Genome STRiP. Finally, when having identified two SVs with a reciprocal coordinate overlap of >80% we removed the one which was predicted in the smaller number of samples.,CNVnator,Genome STRiP,Pindel,Pindel,Genome STRiP,To combine inferred SVs from the different methods, we first merged the predictions of all 3 laboratory strain samples for each method individually. To do so, we defined confidence intervals around the breakpoints, according to the presumed resolution of breakpoint inference for each method (for SVs at nucleotide resolution: +/-10 bp, accounting for misalignments owing to short (<10 bp) indels; for DELLY: 50 bp outwards, 250 bp inwards; for CNVnator: 1 kb outwards, 400 bp inwards). Calls with overlapping confidence intervals at both SV start and end coordinates were combined into a single event. Second, we filtered the individual lists. For DELLY, we required that on average at least 20 read pairs with a mapping quality of at least 20 supported the event in the individual samples. For Pindel as well as CNVnator we did not apply any further filtering. As third step, we defined confidence intervals for each of the filtered lists (intervals as above), and merged the entire SV set based on these confidence intervals. To increase the positive predictive value we filtered out SVs which were predicted by 
a single method only. Finally, when having identified two SVs with a reciprocal coordinate overlap of >80% we removed the one which was predicted in the smaller number of samples.,To combine inferred SVs from the different methods, we first merged the predictions of all 39 samples for each method individually. To do so, we defined confidence intervals around the breakpoints, according to the presumed resolution of breakpoint inference for each method (for SVs at nucleotide resolution: +/-10 bp, accounting for misalignments owing to short (<10 bp) indels; for DELLY: 50 bp outwards, 100 bp inwards; for CNVnator: 1 kb outwards, 400 bp inwards). Calls with overlapping confidence intervals at both SV start and end coordinates were combined into a single event. (For Genome STRiP, a method that performs population-inference of SVs in several samples at once, this merging step was not necessary.) Second, we filtered the individual lists. For DELLY, we required that on average at least 3 read pairs with a mapping quality of at least 20 supported the event in the individual samples. For Genome STRiP, we only considered SVs inferred with more','Merging,Sequencing,Sequencing,Sequencing,Sequencing,Sequencing,Merging,Merging,Merging,Sequencing,Sequencing,Sequencing','Merging,Illumina GAIIx/HiSeq 2000,Illumina GAIIx,Illumina GAIIx,Illumina GAIIx/HiSeq 2000,Illumina GAIIx/HiSeq 2000,Merging,Merging,Merging,Illumina GAIIx/HiSeq 2000,Illumina GAIIx,Illumina GAIIx','Release 5','GCF_000001215.1');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd206','9606','Homo sapiens','Human','24827138','Nagirnaja_et_al_2013','Case-Control',null,'The study aimed to determine the contribution of copy number variants in the genetic etiology of recurrent miscarriage (RM; defined as â‰¥3 consecutive miscarriages before gestational week 22).','Other,Other,Probe signal intensity','QuantiSNP, PennCNV','qPCR,qPCR,SNP array','Custom DNA primers and Applied Biosystems pre-designed TaqMan Copy Number reference assay,Applied Biosystems pre-designed TaqMan Copy Number assays,Illumina Human370CNV-Quad','GRCh37.p13,GRCh38,NCBI36','GCA_000001405.14,GCA_000001405.15,GCF_000001405.12');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd208','9606','Homo sapiens','Human','24281369','Helbig_et_al_2013','Case-Set',null,'Genetics and particularly Copy Number Variations (CNVs) have emerged as significant genetic risk factors for epilepsy. This study aimed to clarify the relevance of CNVs in patients with unclassified epilepsies and complex phenotypes. In total 222 patients from three European countries, including patients with structural lesions on magnetic resonance imaging (MRI), dysmorphic features, and multiple congenital anomalies (MCA), were clinically evaluated and screened for CNVs. Array CGH uncovered 88 rare CNV). We conclude that Genome-wide screening methods for rare CNVs may provide clues for the genetic etiology in patients with a broad range of epilepsies.','SNP genotyping analysis,SNP genotyping analysis,Probe signal intensity,SNP genotyping analysis',null,'SNP array,SNP array,Oligo aCGH,SNP array','Affymetrix 6.0,Affymetrix 6.0,Agilent 105K/180K,Affymetrix 6.0','GRCh37,GRCh37.p13,GRCh38,NCBI36','GCA_000001405.14,GCA_000001405.15,GCF_000001405.12,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd209','9606','Homo sapiens','Human','24192839','Pang_et_al_2013b','Control Set',null,'We observed that current high-throughput sequencing approaches only detected a fraction of the full size-spectrum of insertions, deletions and copy number variants when compared to a previously published, Sanger sequenced human genome. The sensitivity for detection was the lowest in the 100-10,000bp size range, and at DNA repeats, with copy number gains harder to delineate than losses. We discuss strategies for discovering the full spectrum of genetic variation necessary for disease association studies.','Paired-end mapping,Split read mapping,Read depth',null,'Sequencing,Sequencing,Sequencing','Complete Genomics,Complete Genomics,Complete Genomics','GRCh37,GRCh38','GCA_000001405.15,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd21','9606','Homo sapiens','Human','18421352','Wheeler_et_al_2008','Control Set',null,'The association of genetic variation with disease and drug response, and improvements in nucleic acid technologies, have given great optimism for the impact of ''genomic medicine''. However, the formidable size of the diploid human genome, approximately 6 gigabases, has prevented the routine application of sequencing methods to deciphering complete individual human genomes. To realize the full potential of genomics for human health, this limitation must be overcome. Here we report the DNA sequence of a diploid genome of a single individual, James D. Watson, sequenced to 7.4-fold redundancy in two months using massively parallel sequencing in picolitre-size reaction vessels. This sequence was completed in two months at approximately one-hundredth of the cost of traditional capillary electrophoresis methods. Comparison of the sequence to the reference genome led to the identification of 3.3 million single nucleotide polymorphisms, of which 10,654 cause amino-acid substitution within the coding sequence. In addition, we accurately identified small-scale (2-40,000 base pair (bp)) insertion and deletion polymorphism as well as copy number variation resulting in the large-scale gain and loss of chromosomal segments ranging from 26,000 to 1.5 million base pairs. Overall, these results agree well with recent results of sequencing of a single individual by traditional methods. However, in addition to being faster and significantly less expensive, this sequencing technology avoids the arbitrary loss of genomic sequences inherent in random shotgun sequencing by bacterial cloning because it amplifies DNA in a cell-free system. As a result, we further demonstrate the acquisition of novel human sequence, including novel genes not previously identified by traditional genomic sequencing. This is the first genome sequenced by next-generation technologies. Therefore it is a pilot for the future challenges of ''personalized genome sequencing''.','Probe signal intensity,Probe signal intensity,Probe signal intensity,Read depth','CGH_Analytics_Software,CGH_Analytics_Software','Oligo aCGH,Oligo aCGH,Oligo aCGH,Sequencing','Nimblegen HD2 Microarray,Agilent-014584 Human Genome 244K CGH Microarray (Alpha Test),Agilent-014584 Human Genome 244K CGH Microarray (Alpha Test),Roche 454','GRCh37,GRCh37.p13,GRCh38,NCBI36','GCA_000001405.14,GCA_000001405.15,GCF_000001405.12,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd210','9606','Homo sapiens','Human',null,'Blake_et_al_2014','Case-Set',null,'Analysis of structural variation in a patient with neurodevelopmental disease and balanced chromosomal abnormalities. We used mate-pair sequencing to investigate several variants at basepair resolution and to evaluate them as possible genetic causes of the observed disorder.','Sequence alignment,Split read and paired-end mapping',null,'Sequencing,Sequencing','Sanger,Illumina GA II','GRCh37,GRCh38','GCA_000001405.15,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd211','9606','Homo sapiens','Human',null,'Campbell_et_al_2014','Case-Set',null,'Prospective analysis of 100 deletion CNVs to detect somatic mosaicism in parents of children with genomic deletions.','Manual observation',null,'Sequencing','Sanger Sequencing','GRCh37,GRCh38','GCA_000001405.15,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd212','9606','Homo sapiens','Human','25503493','Uddin_et_al_2014','Control Set',null,'We have genotyped a large population control set (1,000 individuals from our Ontario Population Genomics Platform (OPGP)) using the Affymetrix CytoScan HD microarray comprising 2.7 million probes. Four independent algorithms were applied to detect and assess high confidence CNVs. Reproducibility and validations were quantified using sample replicates and Quantitative-PCR (QPCR), respectively.','Probe signal intensity','ChAS','SNP array','Affymetrix CytoScan HD 2.7M array','GRCh37,GRCh38','GCA_000001405.15,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd213','9606','Homo sapiens','Human','24956385','Mokhtar_et_al_2014','Control Set',null,'Copy number variation (CNV) has been recognized as a major contributor to human genome diversity. It plays an important role in determining phenotypes and has been associated with a number of common and complex diseases. However CNV data from diverse populations is still limited. Here we report the first investigation of CNV in the indigenous populations from Peninsular Malaysia. We genotyped 34 Negrito genomes from Peninsular Malaysia using the Affymetrix SNP 6.0 microarray and identified 48 putative novel CNVs, consisting of 24 gains and 24 losses, of which 5 were identified in at least 2 unrelated samples. These CNVs appear unique to the Negrito population and were absent in the DGV, HapMap3 and Singapore Genome Variation Project (SGVP) datasets.','Probe signal intensity','Affymetrix GTC, Birdsuite and iPattern','SNP array','Affymetrix Genome-Wide Human SNP Array 6.0','GRCh37.p13,GRCh38,NCBI36','GCA_000001405.14,GCA_000001405.15,GCF_000001405.12');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd214','9606','Homo sapiens','Human','26432245','1000_Genomes_Consortium_Phase_3','Control Set','http://www.1000genomes.org/','This study contains the structural variants from the combined release set which contains more than 79 million variant sites and includes not just biallelic snps but also indels, deletions, complex short substitutions and other structural variant classes. It is based on data from 2504 unrelated individuals from 26 populations around the world.','Read depth and paired-end mapping',null,'Sequencing','Multiple platforms','GRCh37,GRCh38,GRCh38.p2','GCA_000001405.15,GCA_000001405.17,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd215','9606','Homo sapiens','Human','23714750','GoNL','Control Set','http://www.nlgenome.nl/','GoNL is a whole-genome-sequencing project in a representative sample consisting of 250 trio-families from all provinces in the Netherlands, which aims to characterize DNA sequence variation in the Dutch population. The parent-offspring trios include adult individuals ranging in age from 19 to 87 years (mean=53 years; SD=16 years) from birth cohorts 1910-1994','Sequence alignment','Genome Analysis Toolkit (GATK) BreakDancer, Pindel,24Â GenomeSTRiP, CNVnator, 123SV and DWAC-seq','Sequencing','Illumina HiSeq 2000','GRCh37,GRCh38','GCA_000001405.15,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd216','9606','Homo sapiens','Human','25666259','Kasak_et_al_2014','Case-Control',null,'The study aimed to determine the contribution of copy number variants in the genetic etiology of normal and complicated pregnancies. The samples represented (i) maternal blood DNA drawn from healthy pregnant women during 1st or 2nd trimester and (ii) maternal and paternal blood DNA drawn at term pregnancy cases representing normal and complicated gestations (severe preeclampsia, PE; gestational diabetes, GD; small-for-gestational age newborn, SGA; large-for-gestational age newborn, LGA). We performed CNV calling based on genome-wide genotyping dataset (Illumina HumanOmniExpress-24-v1 BeadChip) by applying three algorithms, QuantiSNP, GADA (Genome Alteration Detection Algorithm) and CNstream in parallel. The acquired CNV calls were merged with HD-CNV (Hotspot Detector for Copy Number Variants) program and only the CNVs predicted by at least two programs, were considered in subsequent analysis.','Probe signal intensity','QuantiSNP, GADA, CNstream','SNP array','Illumina HumanOmniExpress Beadchip','GRCh37,GRCh38','GCA_000001405.15,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd217','9606','Homo sapiens','Human',null,'Besenbacher_et_al_2014','Control Set',null,'Sequencing of Danish parent-offspring trios to determine genomic variation within the Danish population. First release comprises of ten trios sequenced to 50X using libraries of insert sizes from 180nt to 800nt.','de novo sequence assembly','SoapAsmVar','Sequencing','Illumina HiSeq 2000','GRCh37','GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd218','9606','Homo sapiens','Human','25585696','Simpson_et_al_2015','Case-Set',null,'An-exploratory genome-wide copy number variant (CNV) study was performed in 127 independent cases with specific language impairment (SLI) and their first-degree relatives (385 individuals).','Probe signal intensity,Probe signal intensity',null,'SNP array,qPCR','Illumina HumanOmniExpress-12v1 Beadchip,Unspecified','GRCh37','GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd219','9606','Homo sapiens','Human','26432246','1000_Genomes_Consortium_Phase_3_Integrated_SV','Control Set',null,'Structural variations discovered from the phase3 sequence data of the 1000 Genomes Project.','Read depth and paired-end mapping,Split read mapping,Paired-end mapping,Paired-end mapping,Paired-end mapping,Read depth,Merging,Read depth,Read depth and paired-end mapping','Genome STRiP,Pindel,BreakDancerMax,DELLY,NumtSÂ ,SSF,CNVnator,Melt','Sequencing,Sequencing,Sequencing,Sequencing,Sequencing,Sequencing,Merging,Sequencing,Sequencing','Illumina Genome Analyzer,Genome Analyzer II,Genome Analyzer IIx,HiSeq 2000,Illumina Genome Analyzer,Genome Analyzer II,Genome Analyzer IIx,HiSeq 2000,Illumina Genome Analyzer,Genome Analyzer II,Genome Analyzer IIx,HiSeq 2000,Illumina Genome Analyzer,Genome Analyzer II,Genome Analyzer IIx,HiSeq 2000,Illumina Genome Analyzer,Genome Analyzer II,Genome Analyzer IIx,HiSeq 2000,Illumina Genome Analyzer,Genome Analyzer II,Genome Analyzer IIx,HiSeq 2000,Merging,Illumina Genome Analyzer,Genome Analyzer II,Genome Analyzer IIx,HiSeq 2000,Illumina Genome Analyzer,Genome Analyzer II,Genome Analyzer IIx,HiSeq 2000','GRCh37.p4,GRCh38.p3','GCA_000001405.18,GCA_000001405.5');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd22','9606','Homo sapiens','Human','17803354','Levy_et_al_2007','Control Set','http://huref.jcvi.org/','Presented here is a genome sequence of an individual human. It was produced from 32 million random DNA fragments, sequenced by Sanger dideoxy technology and assembled into 4,528 scaffolds, comprising 2,810 million bases (Mb) of contiguous sequence with approximately 7.5-fold coverage for any given region. We developed a modified version of the Celera assembler to facilitate the identification and comparison of alternate alleles within this individual diploid genome. Comparison of this genome and the National Center for Biotechnology Information human reference assembly revealed more than 4.1 million DNA variants, encompassing 12.3 Mb. These variants (of which 1,288,319 were novel) included 3,213,401 single nucleotide polymorphisms (SNPs), 53,823 block substitutions (2-206 bp), 292,102 heterozygous insertion/deletion events (indels)(1-571 bp), 559,473 homozygous indels (1-82,711 bp), 90 inversions, as well as numerous segmental duplications and copy number variation regions. Non-SNP DNA variation accounts for 22% of all events identified in the donor, however they involve 74% of all variant bases. This suggests an important role for non-SNP genetic alterations in defining the diploid genome structure. Moreover, 44% of genes were heterozygous for one or more variants. Using a novel haplotype assembly strategy, we were able to span 1.5 Gb of genome sequence in segments .200 kb, providing further precision to the diploid nature of the genome. These data depict a definitive molecular portrait of a diploid human genome that provides a starting point for future genome comparisons and enables an era of individualized genomic information.','Probe signal intensity,SNP genotyping analysis,SNP genotyping analysis,SNP genotyping analysis,SNP genotyping analysis,SNP genotyping analysis,SNP genotyping analysis,Probe signal intensity,SNP genotyping analysis,Sequence alignment,Merging','CNVFinder,CNAG,CNAG,CNVFinder,CNVFinder,dCHIP,dCHIP,CGH_Analytics_Software,QuantiSNP','Oligo aCGH,SNP array,SNP array,SNP array,SNP array,SNP array,SNP array,Oligo aCGH,SNP array,Sequencing,Merging','Not Reported,[Mapping250K_Sty] Affymetrix Mapping 250K Sty2 SNP Array,[Mapping250K_Nsp] Affymetrix Mapping 250K Nsp SNP Array,[Mapping250K_Nsp] Affymetrix Mapping 250K Nsp SNP Array,[Mapping250K_Sty] Affymetrix Mapping 250K Sty2 SNP Array,[Mapping250K_Sty] Affymetrix Mapping 250K Sty2 SNP Array,[Mapping250K_Nsp] Affymetrix Mapping 250K Nsp SNP Array,Not Reported,Illumina HumanHap650Y Genotyping BeadChip,Sanger Sequencing,Not Reported','GRCh37,GRCh37.p13,GRCh38,NCBI36','GCA_000001405.14,GCA_000001405.15,GCF_000001405.12,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd220','9606','Homo sapiens','Human',null,'Pettigrew_et_al_2015','Case-Control',null,'We screened 86 children with language impairment and/or a family history of dyslexia for copy number variation (CNV). Participants were recruited as part of a longitudinal family study to investigate language and reading development in early childhood, and were assessed over a period of approximately 5 years. DNA was genotyped on an Illumina Human OmniExpress-24 platform, and CNVs predicted using QuantiSNP and PennCNV. Reported CNVs span at least three consecutive genetic variants, have a confidence score > 10, and were predicted by both algorithms.','SNP genotyping analysis',null,'SNP array','Illumina Human OmniExpress-24','GRCh37,GRCh38','GCA_000001405.15,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd221','9606','Homo sapiens','Human','25853576','Palta_et_al_2015','Control Set',null,'The study aimed to: i) call copy number variants (CNVs) from Yoruban and Estonian family trios ii) phase called CNV regions in families; iii) determine allelic variability in unambiguously phased CNV regions.','Probe signal intensity,Probe signal intensity','QuantiSNP, PennCNV,QuantiSNP, PennCNV','SNP array,SNP array','Illumina Infinium Human370CNV-Quad DNA Analysis BeadChip (Illumina 370K),Illumina Infinium Human1M-Duo DNA Analysis BeadChip (Illumina 1M)','NCBI36','GCF_000001405.12');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd222','9606','Homo sapiens','Human','25966632','Franke_et_al_2015','Case-Control',null,'Segmental duplications (SD) comprise about 5% of the human genome and are enriched for immune genes. SD loci often show copy numbers variations (CNV), which are difficult to tag with genotyping methods. CNV in the FcÎ³-receptor region (FCGR) has been suggested to be associated with rheumatic diseases. The objective of this study was to delineate association of FCGR-CNV with rheumatoid arthritis (RA), celiac disease and Inflammatory bowel disease incidence.','SNP genotyping analysis',null,'SNP array','Illumina Immunochip','GRCh37','GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd223','9913','Bos taurus','Cow','26317361','Boussaha_et_al_2015','Control Set',null,'In this study, we performed a large scale study to investigate large SVs (> 50 bp) in cattle by sequencing the whole-genome of 62 bulls from the three major dairy breeds in France (Holstein, Montbeliarde and Normande breeds). SV search was performed using Pindel softaware. We report the identification at a single-nucleotide resolution of 6,426 putative large SVs corresponding to 3,138 large deletions, 1,061 tandem duplications and 2,227 inversions.','Split read and paired-end mapping',null,'Sequencing','Illumina HiSeq2000','Bos_taurus_UMD_3.1','GCF_000003055.4');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd224','9606','Homo sapiens','Human','25118596','Suktitipat_et_al_2014','Control Set','http://thaicnv.icbs.mahidol.ac.th/thaicnv','Thai population control set (3,017 individuals from 7 GWAS studies) using the Illumina SNP array','Probe signal intensity,Probe signal intensity,Probe signal intensity',null,'SNP array,SNP array,SNP array','Illumina HumanHap 550,Illumina Human OmniExpress,Illumina HumanHap 610','NCBI36','GCF_000001405.12');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd225','9606','Homo sapiens','Human','26899349','Magnusson_et_al_2016','Collection',null,'The study is based on CNV scorings from genome wide genotyping (700K SNPs) of blood DNA available on both members of 38 MZ twin pairs. They were identified while undertaking genotyping of almost 10 000 twins in the Swedish TwinGene project. Both members of the 38 MZ-pair were accidentally included due to initial misclassification of zygosity (based on similarity questions). We aimed to investigate occurrence of potential CNV differences between the members of the 38 pairs and thus to test the robustness of the assumption of 100% shared genetic variation in MZ in the classical twin model from the CNV perspective. We find that systemic CNV differences in MZ twins are rare with only one single finding validating with independent methodology. Interestingly, this CNV is a deletion located in the NRXN1 gene encoding Neurexin1 and there covers a small exon. NRXN1 is among the top genes for CNV involvement in development of schizophrenia, cognitive performance, autism and Touretteâ€™s syndrome. Despite this there was no evidence of any mental/cognitive consequences of the deletion in the affected pair.','SNP genotyping analysis,Probe signal intensity',null,'SNP array,qPCR','Illumina OmniExpress bead chip,Not reported','GRCh37','GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd226','9606','Homo sapiens','Human','26893613','Zlotina_et_al_2016','Case-Set',null,'High-resolution molecular cytogenetic analysis of a patient with ring chromosome 18 syndrome combined with a severe congenital subaortic stenosis','Curated,Curated,Curated','CytoGenomics Software (v 3.0.1.1, Agilent Technologies),7500 Software v2.0.6 (Life technologies), ÃŽÂ”ÃŽÂ”Ct method','Oligo aCGH,FISH,qPCR','Agilent SurePrint G3 Human CGH 8x60K,ToTelVysion Probe Kit (Abbott/Vysis),SYBR Green technology','NCBI36','GCF_000001405.12');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd228','9606','Homo sapiens','Human','27124303','Ansari_et_al_2016','Case-Set',null,'Genetic Analysis of Individuals with Aniridia or Gillespie Syndrome','Other',null,'Oligo aCGH','Roche Nimblegen 135k and Agilent targeted','NCBI36','GCF_000001405.12');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd229','9606','Homo sapiens','Human','26490036','Fakhro_et_al_2015','Case-Control',null,'CNVs were detected in 97 Qatari individuals, iuncluding 47 with type 2 diabetes, using a combination of sequencing and array technologies.','Merging,Read depth,Other,Probe signal intensity,Other','CNVPartition,cnMOPS,Unknown,QuantiSNP','Merging,Sequencing,Sequencing,SNP array,SNP array','Merging,HiSeq 2000,HiSeq 2000,Omni2.5M,Omni2.5M','GRCh37','GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd231','9606','Homo sapiens','Human',null,'WONG_et_al_2016','Collection',null,'Deep whole genome sequencing on 48 individuals with Juvenile Idiopathic Arthritis','Split read and paired-end mapping',null,'Sequencing','Illumina HiSeq X Ten','GRCh37','GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd232','9606','Homo sapiens','Human','28231309','Blanco-Kelly_et_al_-2017','Case-Set',null,'Chromosomal deletions at 11p13 deletions are a frequent cause of congenital Aniridia, a rare pan-ocular genetic disease, as well as the related WAGR syndrome. In this study, we have developed a customized targeted CGH array for the WAGR locus on chr11p13-14. We have performed a comprehensive CNV analysis of a large cohort of Spanish patients with aniridia, WAGR syndrome and other related ocular malformations. Here, we report new data from 8 subjects carrying structural variants with different sizes in this locus.','Probe signal intensity,Probe signal intensity','Coffalyser software,Genomic Workbench Standard Edition software (Agilent Technologies) using the Aberration Detection Method 2 (ADM-2)','MLPA,Oligo aCGH','Salsa P219; MRC Holland,Agilent 15K aCGH','GRCh37','GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd233','9606','Homo sapiens','Human',null,'luo_et_al_2017b','Somatic',null,'hepG2 cell line WGS','Read depth and paired-end mapping',null,'Sequencing','xten','GRCh37','GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd234','9913','Bos taurus','Cow',null,'Mesbah-Uddin_et_al_2017','Somatic',null,'We scanned the whole-genome sequences (WGS) of 175 dairy cattle from three dairy cattle breeds, e.g. 67 Holstein, 27 Jersey, and 81 Nordic Red cattle, to discover large deletions. WGS reads were aligned to the bovine reference genome assembly UMD3.1 using BWA, and deletions were detected and genotyped using GenomeSTRiP-2.0. Deletion calls were validated using 777K BovineHD-chip intensity data, targeted breakpoint-assembly, and PCR. The population-genetic properties and functional impacts of the identified deletions were then analyzed.','Read depth and paired-end mapping,Local sequence assembly,Genotyping,Probe signal intensity,Other','Genome STRiP-2.0,TIGRA, BLAT,Genome STRiP-2.0,Genome STRiP-2.0,Sequencher 5.4.6','Sequencing,Sequencing,Sequencing,SNP array,PCR','Illumina HiSeq 2000,Illumina HiSeq 2000,Illumina HiSeq 2000,777K BovineHD Genotyping BeadChip,3500xL Genetic Analyzers','Bos_taurus_UMD_3.1','GCF_000003055.4');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd236','9606','Homo sapiens','Human',null,'Kurtas_et_al_2018','Collection',null,'By-whole genome sequencing (WGS) in three unrelated families, we demonstrated that in one parent of each family a balanced chromothripsis was present causing a genomic imbalance in the index case consisting in a deletion and a non-contiguous duplication within 3q22.1-q26.31 in case 1, a simple two-way reciprocal translocation t(6;14) in case 2, and a complex rearrangement involving chromosomes 6, 7 and 15 in case 3.','Merging,Manual observation,Manual observation,Read depth and paired-end mapping,Merging,Merging',null,'Merging,PCR,Sequencing,Sequencing,Oligo aCGH,Merging','Merging,Other,Sanger Sequencing,Illumina''s TruSeq DNA PCR-Free,Agilent SurePrint G3 Human CGH,Merging','GRCh37','GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd237','3847','Glycine max','Soybean',null,'Lee_et_al_2018','Collection',null,'Large scale insertions (>=500bp) were predicted between cultivated and wild soybean.','de novo sequence assembly,de novo sequence assembly,de novo sequence assembly',null,'Sequencing,Sequencing,Sequencing','Illumina HiSeq 2000,Illumina HiSeq 2000,Illumina HiSeq 2000','Glycine_max_v2.0','GCA_000004515.3');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd239','9031','Gallus gallus','Chicken',null,'Tuanhui_et_al_2019','Collection',null,'A 52-bp indel at ï¿½ï¿½ï¿½477 to ï¿½ï¿½ï¿½425 (ATG as +1) of chicken QPCTL.','Genotyping','Dindel','Sequencing','illumina hiseq 4000','Gallus_gallus-5.0','GCF_000002315.4');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd24','9606','Homo sapiens','Human','17666407','de_Smith_et_al_2007','Control Set',null,'The discovery of copy number variation in healthy individuals is far from complete, and owing to the resolution of detection systems used, the majority of loci reported so far are relatively large ( approximately 65%>10 kb). Applying a two-stage high-resolution array comparative genomic hybridization approach to analyse 50 healthy Caucasian males from northern France, we discovered 2208 copy number variants (CNVs) detected by more than one consecutive probe. These clustered into 1469 CNV regions (CNVRs), of which 721 are thought to be novel. The majority of these are small (median size 4.4 kb) and most have common boundaries, with a coefficient of variation less than 0.1 for 83% of endpoints in those observed in multiple samples. Only 6% of the CNVRs analysed showed evidence of both copy number losses and gains at the same site. A further 6089 variants were detected by single probes: 48% of these were observed in more than one individual. In total, 2570 genes were seen to intersect variants: 1284 in novel loci. Genes involved in differentiation and development were significantly over-represented and approximately half of the genes identified feature in the Online Mendelian Inheritance in Man database. The biological importance of many genes affected, along with the well-conserved nature of the majority of the CNVs, suggests that they could have important implications for phenotype and, thus, be useful for association studies of complex diseases.','Probe signal intensity','CGH_Analytics_Software','Oligo aCGH','Agilent-015366 Custom Human 244K CGH Microarray','GRCh37,GRCh37.p13,GRCh38,NCBI35','GCA_000001405.14,GCA_000001405.15,GCF_000001405.11,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd240','9031','Gallus gallus','Chicken',null,'Tuanhui_et_al_2019','Collection',null,'A new 99-bp indel at âˆ’4409 to âˆ’4309 (ATG as +1) of chicken CEL.','Genotyping','Dindel','Sequencing','Illumina HiSeq 4000','Gallus_gallus-5.0','GCF_000002315.4');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd241','9031','Gallus gallus','Chicken',null,'Tuanhui_et_al_2019','Collection',null,'A 52-bp indel at ï¿½ï¿½ï¿½477 to ï¿½ï¿½ï¿½425 (ATG as +1) of chicken QPCTL.','Genotyping','Dindel','Sequencing','illumina hiseq 4000','Gallus_gallus-5.0','GCF_000002315.4');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd3','9606','Homo sapiens','Human','18987735','Wang_et_al_2008','Control Set',null,'Here we present the first diploid genome sequence of an Asian individual. The genome was sequenced to 36-fold average coverage using massively parallel sequencing technology. We aligned the short reads onto the NCBI human reference genome to 99.97% coverage, and guided by the reference genome, we used uniquely mapped reads to assemble a high-quality consensus sequence for 92% of the Asian individual''s genome. We identified approximately 3 million single-nucleotide polymorphisms (SNPs) inside this region, of which 13.6% were not in the dbSNP database. Genotyping analysis showed that SNP identification had high accuracy and consistency, indicating the high sequence quality of this assembly. We also carried out heterozygote phasing and haplotype prediction against HapMap CHB and JPT haplotypes (Chinese and Japanese, respectively), sequence comparison with the two available individual genomes (J. D. Watson and J. C. Venter), and structural variation identification. These variations were considered for their potential biological impact. Our sequence data and analyses demonstrate the potential usefulness of next-generation sequencing technologies for personal genomics.','Read depth and paired-end mapping',null,'Sequencing','Illumina Genome Analyzer','GRCh37,GRCh37.p13,GRCh38,NCBI36','GCA_000001405.14,GCA_000001405.15,GCF_000001405.12,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd48','9606','Homo sapiens','Human','15654335','Stefansson_et_al_2005','Control Set',null,'A refined physical map of chromosome 17q21.31 uncovered a 900-kb inversion polymorphism.','BAC assembly',null,'Microsatellite genotyping','PTC-225 (MJ Research)','GRCh37,GRCh38,NCBI34','GCA_000001405.15,GCF_000001405.10,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd49','9606','Homo sapiens','Human','18971310','Gusev_et_al_2009','Control Set',null,'We present GERMLINE, a robust algorithm for identifying segmental sharing indicative of recent common ancestry between pairs of individuals. We use GERMLINE to comprehensively survey hidden relatedness both in the HapMap as well as in a densely typed island population of 3000 individuals. We bolster these results by demonstrating novel applications of precise analysis of hidden relatedness for identification and resolution of phasing errors and exposing polymorphic deletions that are otherwise challenging to detect.','SNP genotyping analysis,SNP genotyping analysis,SNP genotyping analysis','GERMLINE,GERMLINE,GERMLINE','SNP array,SNP array,SNP array','[Mapping250K_Nsp] Affymetrix Mapping 250K Nsp SNP Array,Not reported,[Mapping250K_Sty] Affymetrix Mapping 250K Sty2 SNP Array','GRCh37,GRCh37.p13,GRCh38,NCBI36','GCA_000001405.14,GCA_000001405.15,GCF_000001405.12,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd50','9606','Homo sapiens','Human','12058347','Giglio_et_al_2002','Case-Control',null,'Heterozygous submicroscopic inversions involving olfactory receptor-gene clusters mediate the recurrent t(4;8)(p16;p23) translocation.','Probe signal intensity',null,'FISH','Genome-SystemInc','GRCh37,GRCh38,NCBI34','GCA_000001405.15,GCF_000001405.10,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd55','9606','Homo sapiens','Human','17911159','Pinto_et_al_2007','Control Set',null,'Here, we provide an example of how to discover new CNVs from existing genotype data from large-scale genetic epidemiological studies. We also discuss the need to expand surveys of CNV in different population-based cohorts and to apply the information to studies of human variation and disease.','SNP genotyping analysis,SNP genotyping analysis,SNP genotyping analysis,SNP genotyping analysis,SNP genotyping analysis,SNP genotyping analysis','dChip,CNAG,CNAG,dChip,GEMCA,GEMCA','SNP array,SNP array,SNP array,SNP array,SNP array,SNP array','Affymetrix Mapping 250K Sty2 SNP Array,Affymetrix Mapping 250K Nsp SNP Array,Affymetrix Mapping 250K Sty2 SNP Array,Affymetrix Mapping 250K Nsp SNP Array,Affymetrix Mapping 250K Sty2 SNP Array,Affymetrix Mapping 250K Nsp SNP Array','GRCh37,GRCh37.p13,GRCh38,NCBI35','GCA_000001405.14,GCA_000001405.15,GCF_000001405.11,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('estd59','9606','Homo sapiens','Human','20981092','1000_Genomes_Consortium_Pilot_Project','Control Set','http://www.1000genomes.org/','This study contains all structural variation data for the first phase of the 1000 Genomes Project (1K Genomes, 1KG). This includes Deletions, Mobile Element Insertions, Tandem Duplications, and Novel Sequences for both Pilot1 and Pilot2 on all chromosomes. Pilot1 structural variations were determined by low coverage re-sequencing of HapMap samples. Pilot2 structural variations were determined by high coverage re-sequencing of two HapMap trios (YRI and CEU).','Split read mapping,Read depth,Other,Paired-end mapping,Split read mapping,Split read mapping,Split read mapping,Read depth,de novo sequence assembly,de novo sequence assembly,de novo sequence assembly,de novo sequence assembly,de novo sequence assembly,de novo sequence assembly,Other,Split read and paired-end mapping,Split read and paired-end mapping,Split read and paired-end mapping,Split read and paired-end mapping,Split read mapping,de novo sequence assembly,de novo sequence assembly,Other,Paired-end mapping,Paired-end mapping,de novo sequence assembly,de novo sequence assembly,de novo sequence assembly,de novo sequence assembly,de novo sequence assembly,Paired-end mapping,Paired-end mapping,Paired-end mapping,Paired-end mapping,Split read mapping,Other,Paired-end mapping,de novo sequence assembly,Read depth,de novo sequence assembly,de novo sequence assembly,Other,Paired-end mapping,Paired-end mapping,de novo sequence assembly,de novo sequence assembly,Other,de novo sequence assembly,Paired-end mapping,Paired-end mapping,Split read mapping,Paired-end mapping,Paired-end mapping,Paired-end mapping,Probe signal intensity,de novo sequence assembly,Split read mapping,Split read mapping,Paired-end mapping,Read depth,Read depth,Read depth,Paired-end mapping,Paired-end mapping,de novo sequence assembly,de novo sequence assembly,Other','MOSAIK,MAQ,PEM,BLAT,MAQ,Pindel,MAQ,Pindel,MAQ,CORTEX,CORTEX,SOAP & CORTEX,SOAP & CORTEX,SOAP & CORTEX de novo and CORTEX on reference,SOAP & CORTEX de novo and CORTEX on reference,MOSAIK,MOSAIK,MOSAIK,MOSAIK,MOSAIK,SOAP,CORTEX,MOSAIK,MAQ,CORTEX on reference,CORTEX on reference,CORTEX de novo & CORTEX on reference,CORTEX de novo & CORTEX on reference,TIGRA,MAQ,MOSAIK,MOSAIK,MOSAIK,MAQ,Pindel,MAQ,MRFAST,MAQ,NovelSeq & CORTEX on reference,NovelSeq & CORTEX on reference,NovelSeq,SPANNER,MAPREADS,SOAP & CORTEX on reference,SOAP & CORTEX on reference,TIGRA,MAQ,MOSAIK,MRFAST,MAPREADS,SPANNER,SOAP,MOSAIK,BLAT,MAQ,MRFAST,MAQ,MAQ,CORONA,MOSAIK,MAPREADS & SOAP,MAPREADS & SOAP','Sequencing,Sequencing,Digital array,Sequencing,Sequencing,Sequencing,Sequencing,Sequencing,Sequencing,Sequencing,Sequencing,Sequencing,Sequencing,Sequencing,Digital array,Sequencing,Sequencing,Sequencing,Sequencing,Sequencing,Sequencing,Sequencing,PCR,Sequencing,Sequencing,Sequencing,Sequencing,Sequencing,Sequencing,Sequencing,Sequencing,Sequencing,Sequencing,Sequencing,Sequencing,PCR,Sequencing,Sequencing,Sequencing,Sequencing,Sequencing,Sequencing,Sequencing,Sequencing,Sequencing,Sequencing,PCR,Sequencing,Sequencing,Sequencing,Sequencing,Sequencing,Sequencing,Sequencing,Oligo aCGH,Sequencing,Sequencing,Sequencing,Sequencing,Sequencing,Sequencing,Sequencing,Sequencing,Sequencing,Sequencing,Sequencing,PCR','454,Illumina,Affymetrix 6, Illumina 1M, and custom Nimblegen aCGH arrays,454,454,Illumina,Illumina,Illumina,Illumina,Roche 454,Illumina,Roche 454,Illumina,Roche 454,Affymetrix 6, Illumina 1M, and custom Nimblegen aCGH arrays,Roche 454,Illumina,Illumina,Roche 454,Roche 454,Illumina,454,Not reported,Illumina,Illumina,Illumina,Roche 454,Illumina,Roche 454,Not reported,Illumina,Illumina,Illumina,Illumina,Illumina,Not reported,Illumina,Illumina,Illumina,Illumina,Roche 454,Illumina,
Illumina,SOLiD,Illumina,Roche 454,Not reported,Not reported,Illumina,454 GS Titanium,Roche 454,Illumina,ABI SOLiD,Illumina,Custom Nimblegen and Affymetix arrays,Illumina,454,454,Illumina,Illumina,Illumina,Illumina,SOLiD,Illumina,Illumina,ABI SOLiD,Not reported','GRCh37,GRCh37.p13,GRCh38,NCBI36','GCA_000001405.14,GCA_000001405.15,GCF_000001405.12,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd1','9606','Homo sapiens','Human','15895083','Tuzun_et_al_2005','Control Set','http://hgsv.washington.edu','We systematically compared the human genome reference sequence with a second genome (represented by GM15510 fosmid paired-end sequences) to detect intermediate-sized structural variants >8 kb in length','Paired-end mapping,Manual observation,Probe signal intensity,Sequence alignment',null,'Sequencing,PCR,BAC aCGH,Sequencing','Capillary,Not reported,Not reported,Not reported','GRCh37,GRCh37.p13,GRCh38,GRCh38.p2,NCBI35','GCA_000001405.14,GCA_000001405.15,GCA_000001405.17,GCF_000001405.11,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd10','9612','Canis lupus','Wolf','19129542','Nicholas_et_al_2009','Control Set',null,'Here we describe the first systematic and genome-wide analysis of segmental duplications and associated copy number variants (CNVs) in the modern domesticated dog.','Probe signal intensity','RJaCGH','Oligo aCGH','Custom Nimblegen aCGH chip targeted to canFam2 segmental duplications.','CanFam2.0,CanFam3.1','GCF_000002285.1,GCF_000002285.3');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd100','9606','Homo sapiens','Human','25217958','Coe_et_al_2014','Case-Control',null,'Copy Number Variants from 29,083 cases of Developmental Delay and Intellectual Disability from Signature Genomics, and 11,256 Control Samples. This study contains samples in common with [Cooper et al. 2011|/dbvar/studies/nstd54]. Due to analysis differences (see manuscripts) please use the case samples (Sampleset 1) from only one of these submissions. Control sample sets do not overlap and may be combined.','Probe signal intensity,Probe signal intensity,Probe signal intensity',null,'SNP array,SNP array,Oligo aCGH','Affymetrix SNP Array 6.0,Affymetrix SNP Array 6.0,SignatureChip OS','GRCh37,GRCh37.p13,GRCh38,GRCh38.p2,NCBI36','GCA_000001405.14,GCA_000001405.15,GCA_000001405.17,GCF_000001405.12,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd101','9606','Homo sapiens','Human','21844811','ClinGen_Kaminsky_et_al_2011','Case-Set','http://www.clinicalgenome.org','Copy number variation identified through the course of routine clinical cytogenomic testing in postnatal populations. Clinical assertions have been curated as described in [Kaminsky, et al. 2011|/pubmed/21844811]. For additional ClinGen data, please see [nstd37|/dbvar/studies/nstd37/].','Probe signal intensity,Probe signal intensity,Probe signal intensity,Manual observation,Probe signal intensity,SNP genotyping analysis,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Manual observation,Probe signal intensity,Probe signal intensity','Aberration algorithm = ADM-1 with 6.7 threshold; Aberration filter settings: minimum # probes = 4, minimum absolute average log ration = 0.26.,Agilent: Aberration algorithm = ADM-1 with 6.7 threshold; Aberration filter settings: minimum # probes = 5, minimum absolute average log ration = 0.26.','Oligo aCGH,MLPA,Oligo aCGH,FISH,Oligo aCGH,SNP array,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Karyotyping,Oligo aCGH,Oligo aCGH','Agilent ISCA 44K,Not reported,Not reported,Not reported,Not reported,Affymetrix CytoScan HD,Not reported,Not reported,Not reported,ISCA 180k,Not reported,Not reported,Not reported,Not reported','GRCh37,GRCh37.p13,GRCh38,GRCh38.p2,NCBI36','GCA_000001405.14,GCA_000001405.15,GCA_000001405.17,GCF_000001405.12,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd102','9606','Homo sapiens','Human',null,'ClinVar_submitted_variants','Case-Set',null,'Structural Variants submitted to ClinVar by external labs; variants are accompanied by clinical assertions.','Probe signal intensity,Not provided,Probe signal intensity,Not provided,Merging,Not provided,Not provided,Probe signal intensity,Not provided,Not provided,Not provided,Not provided,Not provided,Not provided,Not provided,Multiple,Not provided,Multiple,Not provided,Probe signal intensity,Not provided,Not provided,Not provided,Not provided,Probe signal intensity,Merging,Not provided,Not provided,Read depth,Not provided,Not provided,SNP genotyping analysis,Not provided,Not provided,Multiple,Multiple,Probe signal intensity,Probe signal intensity,Not provided,Not provided,Not provided,Not provided,Merging,Not provided,Not provided,SNP genotyping analysis,Not provided,Merging,Not provided,Not provided,Not provided,Probe signal intensity,Not provided,Not provided,Not provided,Sequence alignment,Not provided,Not provided,Merging,Not provided,Not provided,Not provided,Read depth,SNP genotyping analysis,Read depth,Multiple,Not provided,Probe signal intensity,Not provided,Not provided,Not provided,Not provided,Not provided',null,'Oligo aCGH,Not provided,Oligo aCGH,Not provided,Merging,Not provided,Not provided,Oligo aCGH,Not provided,Not provided,Not provided,Not provided,Not provided,Not provided,Not provided,Multiple,Not provided,Multiple,Not provided,Oligo aCGH,Not provided,Not provided,Not provided,Not provided,Oligo aCGH,Merging,Not provided,Not provided,Sequencing,Not provided,Not provided,SNP array,Not provided,Not provided,Multiple,Multiple,Oligo aCGH,Oligo aCGH,Not provided,Not provided,Not provided,Not provided,Merging,Not provided,Not provided,SNP array,Not provided,Merging,Not provided,Not provided,Not provided,Oligo aCGH,Not provided,Not provided,Not provided,Sequencing,Not provided,Not provided,Merging,Not provided,Not provided,Not provided,Sequencing,SNP array,Sequencing,Multiple,Not provided,Oligo aCGH,Not provided,Not provided,Not provided,Not provided,Not provided','Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported','GRCh37,GRCh37.p13,GRCh38,GRCh38.p2,NCBI36','GCA_000001405.14,GCA_000001405.15,GCA_000001405.17,GCF_000001405.12,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd103','9606','Homo sapiens','Human',null,'LSDB_submitted_variants','Collection',null,'The variants in this study were originally submitted by multiple locus specific mutation databases (LSDBs) to ClinVar. After accessioning at ClinVar the variants were imported to dbVar to be assigned dbVar accessions.','Curated,Curated,Curated,Curated,Curated,Curated,Curated,Curated,Curated',null,'Curated,Curated,Curated,Curated,Curated,Curated,Curated,Curated,Curated','Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported','GRCh37,GRCh38','GCA_000001405.15,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd104','9606','Homo sapiens','Human','25477213','Dumanski_et_al_2014','Control Set',null,'We analyzed the possible causes of loss of chromosome Y (LOY) in blood cells of adult men by studying 6014 men from three independent prospective cohorts: [TwinGene|http://ki.se/en/meb/twingene-and-genomeeutwin] (n=4373) in this study, and [ULSAM|http://www2.pubcare.uu.se/ULSAM/] (n=1153) and [PIVUS|http://www.medsci.uu.se/pivus/] (n=488), both in study [nstd92|http://www.ncbi.nlm.nih.gov/dbvar/studies/nstd92/]. We demonstrated that smoking is associated with LOY in the three cohorts (odds ratios 2.4 â€“4.3), LOY being the most common acquired human mutation in the studied subjects. Our data also suggests that smoking has a transient and dose-dependent mutagenic effect on LOY-status.','SNP genotyping analysis',null,'SNP array','HumanOmniExpress BeadChip','GRCh37','GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd105','9606','Homo sapiens','Human','25745018','Onozawa_et_al_2015','Collection',null,'We reported a previously unrecognized form of polymorphic insertions, termed Templated Sequence Insertion Polymorphism (TSIP), in which the inserted sequence was templated from a distant genomic region. TSIPs can be grouped into two classes based on nucleotide sequence features at the insertion junctions; Class 1 TSIPs show target site duplication (TSD), polyadenylation, and preference for insertion at a 5Ã¢Â€Â™-TTTT/A-3Ã¢Â€Â™ sequence, suggesting a LINE-1 based insertion mechanism, whereas class 2 TSIPs show features consistent with repair of a DNA double strand break by non-homologous end joining. We evaluated whole genome sequence from 52 individuals, and identified 171 TSIPs. Mitochondrial sequences were a frequent template for class 2 insertions, used more commonly than any nuclear chromosome.','Sequence alignment,Sequence alignment',null,'Sequencing,Sequencing','Sanger Sequencing Validation,Complete Genomics Analysis Platform','GRCh37,GRCh38','GCA_000001405.15,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd106','9606','Homo sapiens','Human','24896259','Alsmadi_et_al_2014','Control Set',null,'Comprehensive analysis of two personal genomes of Saudi arabian ancestry from Kuwait','Read depth and paired-end mapping',null,'Sequencing','Illumina HiSeq 2000','GRCh37,GRCh38,GRCh38.p2','GCA_000001405.15,GCA_000001405.17,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd107','9606','Homo sapiens','Human','26484159','John_et_al_2014','Control Set','http://www.sciencedirect.com/science/article/pii/S2213596014001299','Comprehensive analysis of a personal genome of Bedouin ancestry from Kuwait','Read depth and paired-end mapping',null,'Sequencing','Illumina HiSeq 2000','GRCh37,GRCh38,GRCh38.p2','GCA_000001405.15,GCA_000001405.17,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd108','9940','Ovis aries','Sheep','26658352','Tosser-Klopp_et_al_2015','Control Set',null,'A 79-bp deletion detected during whole genome sequencing. Related single-nucleotide variation can be found in [dbSNP|/projects/SNP/snp_viewBatch.cgi?sbid=1062067].','Sequence alignment',null,'Sequencing','Illumina HiSeq2000','Oar_v3.1,Oar_v4.0','GCF_000298735.1,GCF_000298735.2');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd109','9606','Homo sapiens','Human','26503250','Wildschutte_et_al_2015','Collection',null,'Discovery and assembly of polymorphic Alu insertions from 53 human genomes','de novo and local sequence assembly',null,'Sequencing','Illumina HiSeq 2000','GRCh37,GRCh38,GRCh38.p2','GCA_000001405.15,GCA_000001405.17,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd11','9606','Homo sapiens','Human','19651600','Walter_et_al_2009','Tumor vs. Matched-Normal',null,'We performed genome-wide copy number analysis with paired normal and tumor DNA obtained from 86 adult patients with de novo AML using 1.85 million feature SNP arrays.','SNP genotyping analysis','Partek Genomics Suite','SNP array','GPL6801','GRCh37,GRCh37.p13,GRCh38,GRCh38.p2,NCBI36','GCA_000001405.14,GCA_000001405.15,GCA_000001405.17,GCF_000001405.12,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd110','9940','Ovis aries','Sheep','26103004','Wiedemar_et_al_2015','Control Set',null,'Domestic sheep breeds show a broad spectrum of different horn phenotypes. In most modern production breeds sheep are polled (absence of horns), while horns mainly occur in indigenous breeds. Previous studies mapped the responsible locus on ovine chromosome 10. Sequence analysis identified an 1833 bp genomic deletion located in the 3â€™-UTR region of the RXFP2 gene present in horned animals only. The sequenced sheep of the reference assembly was hornless (polled).','Sequence alignment',null,'Sequencing','Not reported','Oar_v3.1,Oar_v4.0','GCF_000298735.1,GCF_000298735.2');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd111','9606','Homo sapiens','Human','26073780','Lou_et_al_2015','Control Set',null,'By using Affymetrix Genome-Wide Human SNP 6.0 array, we detected CNVs from Tibetan samples and identified a Tibetan specific deletion which presents high frequency in Tibetans but low in non-Tibetan populations.','Probe signal intensity,Sequence alignment','Birdsuite and WinXPCNVer','SNP array,Sequencing','Not reported,Not reported','GRCh37,GRCh38,GRCh38.p2','GCA_000001405.15,GCA_000001405.17,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd112','9606','Homo sapiens','Human','26249230','Sudmant_et_al_2015','Collection',null,'To explore the diversity and selective signatures of duplications and deletions in human copy number variation (CNV), we sequenced 236 individuals from 125 distinct human populations. We observed that duplications exhibit fundamentally different population genetic and selective signatures than deletions and are more likely to be stratified between human populations. We find that the proportion of CNV to SNV base pairs is greater among non-Africans than it is among African populations but we conclude that this difference is likely due to unique aspects of non-African population history as opposed to differences in CNV load.','Read depth',null,'Sequencing','Not reported','GRCh37,GRCh38,GRCh38.p2','GCA_000001405.15,GCA_000001405.17,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd113','9606','Homo sapiens','Human','26307204','Polyak_et_al_2015','Case-Set',null,'Copy Number Variant calls for 8,373 individuals (16,828 calls) with intellectual disability/developmental delay and/or autism from Signature Genomics Laboratories LLC. Please note that this study may contain samples in common with [nstd54|/dbvar/studies/nstd54/] and [nstd100|/dbvar/studies/nstd100/]. Due to differences in the analysis please use samplesets from only one of these submissions.','Probe signal intensity',null,'Oligo aCGH','SignatureChip OS','GRCh37,GRCh37.p13,GRCh38,GRCh38.p2,NCBI36','GCA_000001405.14,GCA_000001405.15,GCA_000001405.17,GCF_000001405.12,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd114','60711','Chlorocebus sabaeus','Vervet monkey',null,'Warren_et_al_2015','Collection',null,'We describe our efforts to detect deletion variants unique to the genome of the African green monkey or vervet (Chlorocebus aethiops). In this structural varation study, specifically deletions, we utilized six adult vervets (Chlorocebus aethiops sabaeus), all members of a pedigreed C. a. sabaeus research population. We detect deletion variants with LUMPY then filter these variants with the read depth SV caller CNVnator. Only SVs at a read depth of &lt;1.5 were used for further analysis. The LUMPY deletions for each vervet were filtered to extract deletions ranging from 500bp to 1Mb in size and then genotyped using CNVnator for each of the LUMPY calls.','Read depth,Split read and paired-end mapping,Merging','CNVnator, Svtyper,Lumpy','Sequencing,Sequencing,Merging','Not reported,Not reported,Not reported','Chlorocebus_sabeus 1.1','GCF_000409795.2');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd115','9612,9615,286419','Canis lupus,Canis lupus dingo,Canis lupus familiaris','Dingo,Dog,Wolf','26232412','Decker_et_al_2015','Control Set',null,'We compiled structural variants from diverse modern canids and compared these against variants shared by two ancient clonally transmissible canine tumors.','Split read and paired-end mapping','Delly v0.5.5','Sequencing','Not reported','CanFam3.1','GCF_000002285.3');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd116','9606','Homo sapiens','Human','26526070','Forni_et_al_2015','Control Set',null,'In this study we use high-coverage phase 3 exome sequences of the 1000 Genomes project to infer diploid copy number of the beta-defensin genomic region, a well-studied CNV that carries several beta-defensin genes involved in the antimicrobial response, signalling, and fertility. We also use these data to call sequence variants, a particular challenge given the multicopy nature of the region. We confidently call copy number and sequence variation of the beta-defensin genes on 1285 samples from 26 global populations.','Read depth',null,'Sequencing','Not reported','GRCh37,GRCh38','GCA_000001405.15,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd117','9823','Sus scrofa','Pig','27028052','Long_et_al_2016','Case-Control',null,'We performed a case-control genome-wide CNV association study of Umbilical hernia (UH) on 905 pigs from Duroc, Landrace and Yorkshire breeds using the Porcine SNP60 BeadChip and PennCNV algorithm. We firstly constructed a genomic map comprising 6,193 CNVs that pertain to 737 CNV regions (CNVRs). Then, we identified eight CNVs significantly associated with the risk for UH in the three pig breeds. Six of seven significantly associated CNVs were validated using quantitative real-time PCR. Notably, a rare CNV (CNV14:13030843-13059455) encompassing the NUGGC gene, implicated in human omphalocele and inguinal hernia, was strongly associated with UH (Permutation-corrected P = 0.0015) in Duroc pigs.','SNP genotyping analysis,Probe signal intensity',null,'SNP array,qPCR','Infinium II Multisample assay platform (Illumina),Not reported','Sscrofa10.2','GCF_000003025.5');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd118','9606','Homo sapiens','Human','25848046','Polley_et_al_2015','Control Set',null,'We characterised copy number variation at the DMBT1 gene, which encodes a large glycoprotein known as salivary agglutinin, gp340, muclin or hensin. DMBT1 has a complex copy number variable structure, with two, independent, rapidly mutating copy number variable regions, called CNV1 and CNV2. We analysed the distribution across populations, calculated its mutation rate from analysing pedigrees. We provide evidence suggesting that CNV has been shaped by selection, in particular ability to bind both Streptococcus mutans and hydroxyapatite on the tooth surface.','Manual observation,Manual observation,Probe signal intensity,Probe signal intensity,Probe signal intensity','Manual observation,Manual observation,CNVtools,CNVtools,CNVtools','PCR,PCR,qPCR,Oligo aCGH,qPCR','Not reported,Not reported,Not reported,Not reported,Not reported','GRCh37,GRCh38','GCA_000001405.15,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd119','9913','Bos taurus','Cow','26763170','Menzi_et_al_2016','Case-Set',null,'A transposable element (LTR) insertion in APOB causes cholesterol deficiency (CD) in Holstein cattle. This recessivly inherited mutation leads to a loss of function mutation of the bovine apolipoprotein B (APOB) gene. Homozygous affected animals showed hypocholesterolemia and hypolipoproteinemia and die after a variable period of days to months with idiopathic diarrhea indicating a monogenic recessively inherited fat metabolism disorder (OMIA 001965-9913).','Sequence alignment',null,'Sequencing','Not reported','Bos_taurus_UMD_3.1.1,Btau_5.0.1','GCF_000003055.5,GCF_000003205.7');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd12','9606','Homo sapiens','Human','18252227','Marshall_et_al_2008','Case-Set','http://projects.tcag.ca/autism_500k/','SNP array and karyotyping were used to assess structural abnormalities in 427 unrelated ASD cases.','Merging,Merging,Merging,Merging,Merging,Merging,Merging,Merging,SNP genotyping analysis,SNP genotyping analysis,SNP genotyping analysis,SNP genotyping analysis,Merging,Merging,Merging,Merging,Merging,Merging,Merging,Merging,SNP genotyping analysis,SNP genotyping analysis,SNP genotyping analysis,SNP genotyping analysis,SNP genotyping analysis,SNP genotyping analysis,SNP genotyping analysis,SNP genotyping analysis,Merging,Merging','dCHIP,GEMCA,GEMCA,GEMCA,GEMCA,CNAG,CNAG,CNAG,CNAG,dCHIP,dCHIP,dCHIP','Merging,Merging,Merging,Merging,Merging,Merging,Merging,Merging,SNP array,SNP array,SNP array,SNP array,Merging,Merging,Merging,Merging,Merging,Merging,Merging,Merging,SNP array,SNP array,SNP array,SNP array,SNP array,SNP array,SNP array,SNP array,Merging,Merging','See merged experiments,See merged experiments,See merged experiments,See merged experiments,See merged experiments,See merged experiments,See merged experiments,See merged experiments,[Mapping250K_Sty] Affymetrix Mapping 250K Sty2 SNP Array,Affymetrix GeneChip Early Access Mapping 500K Set Array (250K_Nsp_SNP),[Mapping250K_Nsp] Affymetrix Mapping 250K Nsp SNP Array,Affymetrix GeneChip Mapping 500K Early Access Array (250K_Sty_SNP),See merged experiments,See merged experiments,See merged experiments,See merged experiments,See merged experiments,Not reported,See merged experiments,See merged experiments,[Mapping250K_Sty] Affymetrix Mapping 250K Sty2 SNP Array,Affymetrix GeneChip Early Access Mapping 500K Set Array (250K_Nsp_SNP),Affymetrix GeneChip Mapping 500K Early Access Array (250K_Sty_SNP),[Mapping250K_Nsp] Affymetrix Mapping 250K Nsp SNP Array,[Mapping250K_Nsp] Affymetrix Mapping 250K Nsp SNP Array,Affymetrix GeneChip Mapping 500K Early Access Array (250K_Sty_SNP),Affymetrix GeneChip Early Access Mapping 500K Set Array (250K_Nsp_SNP),[Mapping250K_Sty] Affymetrix Mapping 250K Sty2 SNP Array,See merged experiments,See merged experiments','GRCh37,GRCh37.p13,GRCh38,GRCh38.p2,NCBI35','GCA_000001405.14,GCA_000001405.15,GCA_000001405.17,GCF_000001405.11,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd121','9796','Equus caballus','Horse','27440410','Kader_et_al_2016','Collection',null,'The Copy Number Variation (CNV), as an essential form of genetic variation, has been increasingly recognized as one promising genetic marker in the analyses of animal genomes. Here we used the Equine 70K SNP genotyping array for the genome-wide detection of CNVs in 96 horses from three diverse Chinese breeds, Debao pony (DB), Mongolian (MG) and Yili (YL) horse. A total of 287 CNVs were determined and merged into 122 CNV regions (CNVRs) ranging from 199 bp to 2344 kb in size and distributed in a heterogeneous manner on chromosomes.','SNP genotyping analysis',null,'SNP array','Not reported','EquCab2.0','GCF_000002305.2');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd122','9606','Homo sapiens','Human','26749307','Duyzend_et_al_2015','Case-Control',null,'Copy Number Variants from 459 subjects from families where at least one individual carries a 16p11.2 CNV and 4,092 control samples. Case and control samplesets do not overlap.','Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity','Illumina CNVPartition v3.2.0,Illumina CNVPartition v3.2.0,Illumina CNVPartition v3.2.0,Illumina CNVPartition v3.2.0','SNP array,SNP array,SNP array,SNP array','Illumina 1.2M WTCCC Custom,Illumina Human Omni Express v1,Illumina Human Omni Express v2,Illumina 1.2M WTCCC Custom','GRCh37,GRCh37.p13,GRCh38,GRCh38.p2,NCBI36','GCA_000001405.14,GCA_000001405.15,GCA_000001405.17,GCF_000001405.12,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd123','9615','Canis lupus familiaris','Dog',null,'Beckers_et_al_2016','Collection',null,'In light of improving breeding advice, Belgian dogs were genotyped for genetic disorders in order to perform a frequency estimation of causal mutations.','Sequence alignment,Sequence alignment,Sequence alignment,Sequence alignment',null,'Sequencing,Sequencing,Sequencing,Sequencing','Not reported,Not reported,Not reported,Not reported','CanFam3.1','GCF_000002285.3');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd124','9606','Homo sapiens','Human','26990204','Binder_et_al_2016','Case-Set',null,'We applied whole genome sequencing to a DNA sample from a Multiple Sclerosis (MS) affected patient. This individual had previously been genotyped at multiple SNPs within the MERTK gene and was determined to be homozygous for the MS risk-associated haplotype at this gene locus. To interrogate the MS risk haplotype at this locus in an unbiased way, we sequenced the whole genome of this MS - MERTK homozygous-risk subject. We identified a large number of SNPs and small indels within the risk haplotype, and also identified a retrotransposon insertion within intron 4 (type AluYf4), and also identified an expanded (TA)n(T)m tandem repeat region within intron 1.','Paired-end mapping','Mapping with: BWA MEM, Data Visualization with Integrative Genomics Viewer (IGV) software','Sequencing','Illumina HiSeq 2000','GRCh37,GRCh38','GCA_000001405.15,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd125','9606','Homo sapiens','Human','27552964','Wills_et_al_2016','Somatic',null,'Autosomal dominant polycystic liver disease (ADPLD) and autosomal dominant polycystic kidney disease (ADPKD) is caused by mutations in PRKCSH, SEC63, and LRP5, PKD1, and PKD2. Liver cyst development in these disorders is explained by somatic loss-of-heterozygosity (LOH) of the wild type allele in the developing cyst. We analyzed 24 liver cyst samples from 23 patients using high resolution microarray (homozygosity of autosomes (&gt;3.0Mb) large CNVs (&gt;1.0Mb)). We found frequent LOH in PRKCSH (22/29), and PKD1/PKD2 (2/3). In the total cohort, 12/23 patients harbored abnormalities outside of familiar areas. In individual ADPLD cases, we identified germline events: a 2q13 complex rearrangement resulting in BUB1 haploinsufficiency, a 47XXX karyotype, and LOH on chromosome 3p.','SNP genotyping analysis',null,'SNP array','Not reported','GRCh37,GRCh38,GRCh38.p2','GCA_000001405.15,GCA_000001405.17,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd126','9615','Canis lupus familiaris','Dog','27917507','Turba_et_al_2016','Case-Set',null,'The aims of the study were the following: 1) to establish the allele frequencies of the c.118G&gt;A substitution in the Hovawart breed, 2) to verify the possibility of Modifier, and 3) to establish the cause of drop out effect leading to errors in genetic testing.','Manual observation,Sequence alignment','GeneMapper,SeqScape v2.5','PCR,Sequencing','Not reported,Not reported','CanFam3.1','GCF_000002285.3');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd127','9606','Homo sapiens','Human','27231129','Dumanski_et_al_2016','Somatic',null,'Men have a shorter life expectancy compared with women but the underlying factor(s) are not clear. Late-onset, sporadic Alzheimerâ€™s disease (AD) is a common and lethal neurodegenerative disorder and many germline inherited variants have been found to influence the risk of developing AD. Our previous results show that a fundamentally different genetic variant, i.e. lifetime-acquired loss of chromosome Y (LOY) in blood cells, is associated with all-cause mortality, an increased risk of non-hematological tumors and that LOY could be induced by tobacco smoking. We tested here a hypothesis whether men with LOY are more susceptible to AD and show that LOY is associated with AD in three independent studies of different types. In a case-control study, males with AD diagnosis had higher degree of LOY mosaicism (adjusted odds ratio=2.80, p=0.0184, AD events=606). Furthermore, in two prospective studies men with LOY at blood sampling had greater risk for incident AD diagnosis during follow-up time (hazard ratio HR=6.80, 95% confidence interval (95% CI)=2.16-21.43, AD events=140, p=0.0011). Thus, LOY in blood is associated with risks of both AD and cancer, suggesting a role of LOY in blood cells on disease processes in other tissues, possibly via defective immunosurveillance. As a male-specific risk factor, LOY might explain why males on average live shorter lives than females.','SNP genotyping analysis',null,'SNP array','Illumina Human610-Quad v1.0 BeadChip','GRCh37,GRCh38,NCBI36','GCA_000001405.15,GCF_000001405.12,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd128','9606','Homo sapiens','Human','27654912','Mallick_et_al_2016','Collection',null,'We report high quality genomes from 300 individuals from 142 diverse populations. As part of this study, we generated a comprehensive catalog of short tandem repeat (STR) genotypes. We used this call set to characterize allele frequency spectra, analyze sequence determinants of STR variation, and to identify common loss of function alleles.','Genotyping',null,'Sequencing','Illumina','GRCh37,GRCh38,GRCh38.p2','GCA_000001405.15,GCA_000001405.17,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd129','9606','Homo sapiens','Human','27569546','Guo_et_al_2016','Control Set',null,'We used the Illumina HumanExome array to genotype 753 patients of European descent presenting specifically sporadic thoracic aortic dissection (STAD) and compared them to the genotypes of 2259 controls. SNPs in FBN1, LRP1, and ULK4 were identified to be significantly associated with STAD, and these results were replicated in two independent cohorts. Case-control association study of genomic copy number variation (CNV) on these loci were further performed to test the allele frequency of CNV in patients with that of controls from the dbGaP database. CNV analysis independently confirmed that ULK4 deletions were significantly associated with development of thoracic aortic disease. These results indicate that genetic variations in LRP1 and ULK4 contribute to risk for presenting with an acute aortic dissection. The variant reported here was observed in a control individual.','SNP genotyping analysis','A consensus CNV call list was derived from the combination of three algorithms:  PennCNV (v. 2011), Nexus Copy Number (v. 7.5, Biodiscovery, Inc., Hawthorne, CA), and CNVPartition.','SNP array','Genomic copy number analysis','GRCh37,GRCh38','GCA_000001405.15,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd13','9615','Canis lupus familiaris','Dog','19015322','Chen_et_al_2009','Control Set',null,'Here we report the use of high resolution oligonucleotide (oligo) aCGH to map common CNVs in pedigreed dogs (commonly referred to as â€˜â€˜purebredâ€™â€™).','Sequence alignment,Manual observation,Manual observation,Probe signal intensity','Bioconductor','Sequencing,Southern,PCR,Oligo aCGH','Not reported,Not reported,Not reported,NimbleGen Canis familiaris 385K CanFam2_WG_CGH','CanFam2.0,CanFam3.1','GCF_000002285.1,GCF_000002285.3');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd130','9606','Homo sapiens','Human','27569545','Leppa_et_al_2016','Collection',null,'We analyzed microarray based CNV data from families from the Autism Genetic Resource Exchange (AGRE). We observed a higher burden of large, rare CNVs in individuals with ASD compared to their unaffected siblings, but at a significantly lower level than in simplex families. We also identified a rare ASD and language delay associated de novo deletion overlapping the NR4A2 gene on 2q24.1. Thirty families with previously characterized ASD-associated CNVs showed inheritance pattern, where in 21 of the 30 families, in which at least one affected sibling harbored a major risk CNV, the CNV wasnï¿½ï¿½ï¿½t shared by all affected children within that family, indicating other contributing factors.','Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity','CNVision (PennCNV, QuantiSNP, GNOSIS),CNVision (PennCNV, QuantiSNP, GNOSIS),CNVision (PennCNV, QuantiSNP, GNOSIS),CNVision (PennCNV, QuantiSNP, GNOSIS),CNVision (PennCNV, QuantiSNP, GNOSIS),CNVision (PennCNV, QuantiSNP, GNOSIS)','SNP array,SNP array,SNP array,SNP array,SNP array,SNP array','Illumina HumanHap 550v1,Illumina Omni - 1,Illumina Omni - 1,Illumina Omni - 1,Illumina Omni-2.5,Illumina HumanHap 550v3','GRCh37,GRCh37.p13,GRCh38,GRCh38.p2,NCBI36','GCA_000001405.14,GCA_000001405.15,GCA_000001405.17,GCF_000001405.12,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd131','9913','Bos taurus','Cow','27775157','Keel_et_al_2016','Collection',null,'In this study we identified CNV in a population bulls using low coverage next-generation sequencing data. First, in order to determine a suitable strategy for CNV detection in our data, we compared the performance of three distinct CNV detection algorithms on benchmark CNV data sets and concluded that using the multiple sample read depth approach was the best method for identifying CNV in our sequence. Using this technique, we identified a total of 1,532 copy number variable regions (CNVRs) from genome sequence of 154 purebred sires used in Cycle VII of the Germplasm Evaluation Project (GPE). These bulls represented the seven most popular beef breeds in the United States, Hereford, Charolais, Angus, Red Angus, Simmental, Gelbvieh, and Limousin. The CNVRs covered approximately 7% of the bovine genome and spanned 2,004 protein-coding genes and many known quantitative trait loci (QTL). Gene ontology enrichment analysis showed that they are significantly enriched for terms related to immune system process.','Read depth and paired-end mapping','cn.MOPS','Sequencing','Illumina HiSeq','Bos_taurus_UMD_3.1,Bos_taurus_UMD_3.1.1,Btau_5.0.1','GCF_000003055.4,GCF_000003055.5,GCF_000003205.7');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd132','9606','Homo sapiens','Human','28145423','Walker_et_al_2017','Case-Set',null,'We conducted a genome-wide association analysis of copy number variants (CNVs) with breast or ovarian cancer risk in a cohort of 2500 BRCA1 pathogenic variant carriers, using Illumina 610k SNP array data from a previously study (PMID: [20852631|http://www.ncbi.nlm.nih.gov/pubmed/20852631]). This study highlighted the need to verify CNVs in vitro, but also provides evidence that experimentally validated CNVs (with plausible biological consequences) can modify risk of breast or ovarian cancer in BRCA1 pathogenic variant carriers.','Genotyping','CNVs called by at least two algorithms (CNVPartition, GNOSIS, PennCNV, QuantiSNP).','SNP array','Illumina Infinium 610K array','GRCh37,GRCh37.p13,GRCh38,GRCh38.p2,NCBI36','GCA_000001405.14,GCA_000001405.15,GCA_000001405.17,GCF_000001405.12,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd133','9606','Homo sapiens','Human','27841880','Redin_et_al_2016','Case-Set',null,'Detailed breakpoint mapping of 248 balanced chromosomal rearrangements at nucleotide resolution using sequencing in subjects with congenital anomalies, based on initially provided karyotpes.','Merging,Split read and paired-end mapping,Split read and paired-end mapping,Split read mapping',null,'Merging,Sequencing,Sequencing,Sequencing','Not reported,Not reported,Not reported,Not reported','GRCh37,GRCh38,GRCh38.p2','GCA_000001405.15,GCA_000001405.17,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd134','7227','Drosophila melanogaster','Fruit fly','27928499','Gilks_et_al_2016','Collection','http://www.sussex.ac.uk/lifesci/morrowlab/','As part of a study into the molecular genetics of sexual selection, we used next-generation sequencing to obtain data on genomic variation in an out-bred Drosophila melanogaster population sample. As expected, two in-house reference line individuals had few mutations compared to the reference assembly, and a high level of genotype concordance. Furthermore, the LHM hemiclones have millions of high-quality heterozygous genotypes which exhibit genomic position and allele frequency distributions similar to that of other D.melanogaster population samples, and with generally low singleton and genotype drop-out rates. These data can be used to compare the performance of next-generation sequencing, and for population genomics of a diploid model organism from haploid genotype data.','Read depth and paired-end mapping','Genomestrip/2.0','Sequencing','Illumina 2500','Release 6 plus ISO1 MT','GCF_000001215.4');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd135','9913','Bos taurus','Cow',null,'Karimi_et_al_2016','Collection',null,'After merging the overlapping CNVs, a total of 221 CNV regions were identified encompassing 36.4 Mb or 1.44% of the bovine autosomal genome. The length of the CNV regions ranges from 3.5 to 2252.8 Kb with an average of 163.8 Kb. These regions included 147 loss (66.52%) and 74 gain (33.48%) events containing a total of 638 annotated Ensembl genes. Gene ontology analysis revealed that most of the genes in the CNV regions were involved in the environmental responses, disease susceptibility and immune system functions. Furthermore, 586 of these genes corresponded to the human orthologous genes which involved in a wide range of biological functions. Altogether, 73% of the 221 CNV regions overlapped either completely or partially with those previously reported in the other cattle studies. Moreover, novel CNV regions involved several QTL related to adaptative traits of Iranian indigenous cattle.','SNP genotyping analysis',null,'SNP array','Not reported','Bos_taurus_UMD_3.1,Bos_taurus_UMD_3.1.1,Btau_5.0.1','GCF_000003055.4,GCF_000003055.5,GCF_000003205.7');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd137','9606','Homo sapiens','Human','27895111','Huddleston_et_al_2016','Collection',null,'In an effort to more fully understand the full spectrum of human genetic variation, we generated deep single-molecule, real-time (SMRT) sequencing data from two haploid human genomes. Using an assembly-based approach (SMRT-SV), we systematically assessed each genome independently for structural variants (SVs) and indels resolving the sequence structure of 461,553 genetic variants from 2 bp to 28 kbp in length. We find that 82% of these variants have been missed as part of analysis of the 1000 Genomes Project. We estimate that this theoretical human diploid differs by as much as ~16 Mbp with respect to the human reference, with long-read sequencing data providing a fivefold increase in sensitivity for genetic variants ranging in size from 7 bp to 1 kbp when compared to short-read sequence data.','Genotyping,Local sequence assembly,Local sequence assembly,Genotyping',null,'Sequencing,Sequencing,Sequencing,Sequencing','Not reported,Not reported,Not reported,Not reported','GRCh37,GRCh37.p13,GRCh38','GCA_000001405.14,GCA_000001405.15,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd138','9823','Sus scrofa','Pig',null,'Li_et_al_2016','Control Set',null,'We identified candidate indels in ten pig assemblies using a previously reported methodology. The de novo assembly of ten individuals were separately aligned onto the reference genome (Sscrofa10.2) using the LASTZ tool. The predicted gaps in the pair-wise alignments between the two genome assemblies were extracted using the SOAP software and defined as candidate indels. To filter out spurious indels, we separately aligned the reads onto both the reference genome (Sscrofa10.2) and ten assemblies using the BWA tool, and calculated the read coverage for each candidate indel. Then different criteria were used to validate the candidate indels â‰¤ 50 bp or &gt; 50 bp as previously described.','de novo sequence assembly,de novo sequence assembly,de novo sequence assembly,de novo sequence assembly,de novo sequence assembly,de novo sequence assembly,de novo sequence assembly,de novo sequence assembly,de novo sequence assembly,de novo sequence assembly',null,'Sequencing,Sequencing,Sequencing,Sequencing,Sequencing,Sequencing,Sequencing,Sequencing,Sequencing,Sequencing','Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported','Sscrofa10.2','GCF_000003025.5');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd139','9606','Homo sapiens','Human',null,'Luo_et_al_2017','Case-Set',null,'Gene deletion studies in mice have showed that CatSper is crucial for mammalian male fertility. However, the significance of CatSper for human fertilization remains largely unclear. Therefore, a case only containing CatSper deficiency should help reveal the significance of CatSper on human sperm functions. In this study, we identified a CatSper current-deficient man who is an idiopathic infertile patient with normal sperm morphology, count and initial motility. This patient is an ideal case to study the functional significance of CatSper for fertilization success. Thus human whole-genome resequencing was used to identify novel genetic variations responsible for infertility in the patient.','Paired-end mapping','CNVs were analyzed by Control-FREEC software (Boeva, et al., 2012)','Sequencing','Illumina HiSeq X Ten','GRCh37,GRCh38','GCA_000001405.15,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd14','9606','Homo sapiens','Human','18776910','Cooper_et_al_2008','Control Set',null,'Using independent, sequence based CNV maps, we find that commonly used SNP platforms have limited or no probe coverage for a large fraction of CNVs. Despite this, in 9 samples we inferred 368 CNVs using Illumina SNP genotyping data and experimentally validated over two-thirds of these.','SNP genotyping analysis','HMMSeg','SNP array','Illumina Human1Mv1 DNA Analysis BeadChip (Human1Mv1_C)','GRCh37,GRCh37.p13,GRCh38,GRCh38.p2,NCBI35','GCA_000001405.14,GCA_000001405.15,GCA_000001405.17,GCF_000001405.11,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd140','9606','Homo sapiens','Human','28104618','Fan_et_al_2017','Control Set',null,'We developed a hybrid structural variant assembly (HySA) approach that integrates sequencing reads from next-generation sequencing and single-molecule sequencing technologies to accurately assemble and detect structural variations in human genomes. We applied HySA to a haploid hydatidiform mole genome (CHM1) and a diploid human genome (NA12878) for SV detection.','Local sequence assembly,Local sequence assembly','HySA,HySA','Sequencing,Sequencing','Not reported,Not reported','GRCh37,GRCh37.p13,GRCh38,GRCh38.p2','GCA_000001405.14,GCA_000001405.15,GCA_000001405.17,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd141','9606','Homo sapiens','Human',null,'Rambo-Martin_et_al_2017','Case-Control',null,'One in five people with Down syndrome (DS) are born with an atrioventricular septal defect (AVSD), an incidence 2,000 times higher than in the euploid population. We tested 198 case individuals with DS+AVSD and 211 control individuals with DS and a normal heart using a custom microarray with dense probes tiled on chromosome 21 for array CGH.','Probe signal intensity',null,'Oligo aCGH','Agilent Sureselect 8x60k CGH array','GRCh37,GRCh38','GCA_000001405.15,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd142','9606','Homo sapiens','Human','27995740','Rahbari_et_al_2016','Control Set',null,'Fcï¿½ï¿½ receptors are a family of cell-surface receptors that are expressed by a host of different innate and adaptive immune cells and mediate inflammatory responses by binding the Fc portion of immunoglobulin G (IgG). In humans, five low affinity receptors are encoded by the genes FCGR2A, FCGR2B, FCGR2C, FCGR3A and FCGR3B, which are located in a 82.5kb segmental tandem duplication on chromosome 1q23.3, which shows extensive copy number variation (CNV). We identify several alleles with gene conversion events using fosmid sequencing data.','Sequence alignment',null,'Sequencing','Not reported','GRCh37,GRCh38,NCBI36','GCA_000001405.15,GCF_000001405.12,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd143','9606','Homo sapiens','Human',null,'Shang_et_al_2017','Collection',null,'We identified a group of additional genetic determinants contributing to disease, which provided an accurate molecularï¿½ï¿½diagnosisï¿½ï¿½for the patients with ï¿½ï¿½-thalassemia. Molecularï¿½ï¿½screening test report the diagnostic mutations undetected using routine methods, including pathogenic copy-number variations (CNV) and some variants. Based on the data observed from the 22,260 individuals, we revealed surprisingly high carrier frequencies for hemoglobinopathies in southern China.','Read depth and paired-end mapping',null,'Sequencing','Not reported','GRCh37,GRCh38','GCA_000001405.15,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd144','9598,9606,63221,741158','Homo sapiens,Homo sapiens neanderthalensis,Homo sapiens ssp. Denisova,Pan troglodytes','Chimpanzee,Hominin,Human,Neanderthal',null,'Gardner_et_al_2017','Control Set',null,'Mobile element insertions (MEIs) represent ~25% of all structural variants in human genomes. Moreover, when MEIs disrupt genes, they can influence human traits and diseases. Therefore, MEIs should be fully discovered along with other forms of genetic variation. Here, we describe the Mobile Element Locator Tool (MELT), which was developed as part of the 1000 Genomes Project to perform routine MEI discovery on a population scale. We then used MELT to perform MEI discovery in modern humans, chimpanzees, and ancient (Neanderthal and Denisovan) hominids. Overall, our study provides the most comprehensive map of MEIs to date spanning chimpanzees, ancient hominids, and modern humans.','Split read and paired-end mapping,Split read and paired-end mapping,Split read and paired-end mapping,Split read and paired-end mapping','Mobile Element Locator Tool (MELT),Mobile Element Locator Tool (MELT),Mobile Element Locator Tool (MELT),Mobile Element Locator Tool (MELT)','Sequencing,Sequencing,Sequencing,Sequencing','HiSeq 2000,HiSeq 2000,HiSeq 2000,HiSeq 2000','GRCh37,GRCh38,GRCh38.p2,Pan_troglodytes-2.1.4','GCA_000001405.15,GCA_000001405.17,GCA_000001515.4,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd145','9606','Homo sapiens','Human','28705883','Lu_et_al_2017','Control Set',null,'We used 4.2M NimbleGen CGH array with a pooling reference of 200 Han Chinese to detect genome-wide CNVs in 425 male Han Chinese samples from 28 dialect groups in China. Based on the results, we built a high-quality CNV map which could serve as resources for medical and genetic studies.','Probe signal intensity',null,'Oligo aCGH','Not reported','GRCh37,GRCh38,GRCh38.p2','GCA_000001405.15,GCA_000001405.17,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd146','9615','Canis lupus familiaris','Dog',null,'Switonski_et_al_2017','Collection',null,'The study is focused on searching for genetic markers predisposing to obesity in dogs','Sequence alignment',null,'Sequencing','Not reported','CanFam3.1','GCF_000002285.3');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd149','9606','Homo sapiens','Human',null,'Gambin_et_al_2017','Case-Set',null,'We retrospectively analyzed data from 63,127 patients referred for clinical chromosomal microarray analysis (CMA) at Baylor Genetics laboratories, including 46,755 individuals tested using exon-targeted arrays, from 2007 to 2017. Small CNVs harboring a single gene or two to five non-disease associated genes were identified, and the genes involved were evaluated for a potential disease association.','Curated',null,'Oligo aCGH','Not reported','GRCh37,GRCh38','GCA_000001405.15,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd15','10090','Mus musculus','Mouse','17989247','Cutler_et_al_2007','Control Set',null,'In this study, we have performed the most comprehensive survey to date of CNVs in mice, analyzing the genomes of 42 Mouse Phenome Consortium priority strains. This microarray comparative genomic hybridization (CGH)-based analysis has identified 2094 putative CNVs, with an average of 10 Mb of DNA in 51 CNVs when individual mouse strains were compared to the reference strain C57BL/6J.','Probe signal intensity','GLAD','Oligo aCGH','Agilent-014695 Mouse Genome CGH Microarray 244A (G4415A) (Feature Number version)','GRCm38,GRCm38.p3,MGSCv36,MGSCv37','GCF_000001635.15,GCF_000001635.16,GCF_000001635.20,GCF_000001635.23');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd151','9606','Homo sapiens','Human','27533299','Exome_Aggregation_Consortium_CNVs','Case-Set','http://exac.broadinstitute.org','Copy number variation (CNV) affecting protein-coding genes contributes substantially to human diversity and disease. Here we characterized the rates and properties of rare genic CNVs (&lt;0.5% frequency) in exome sequencing data from nearly 60,000 individuals in the Exome Aggregation Consortium (ExAC) database. For every gene, we empirically estimated an index of relative intolerance to CNVs that demonstrated moderate correlation with measures of genic constraint based on single-nucleotide variation (SNV) and was independently correlated with measures of evolutionary conservation. The ExAC CNV data constitute a critical component of an integrated database spanning the spectrum of human genetic variation, aiding in the interpretation of personal genomes as well as population-based disease studies.','Read depth',null,'Sequencing','Not reported','GRCh37,GRCh38,GRCh38.p2','GCA_000001405.15,GCA_000001405.17,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd16','9606','Homo sapiens','Human','17901297','Korbel_et_al_2007','Control Set',null,'We introduce high-throughput and massive paired-end mapping (PEM), a large-scale genome-sequencing method to identify structural variants (SVs) ~3 kilobases (kb) or larger that combines the rescue and capture of paired ends of 3-kb fragments, massive 454 sequencing, and a computational approach to map DNA reads onto a reference genome.','Manual observation,Manual observation,Paired-end mapping,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity',null,'PCR,FISH,Sequencing,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH','Not reported,Not reported,454,NimbleGen Homo sapiens HG18 Whole Genome CGH Tiling Set (6 of 8),NimbleGen Homo sapiens HG18 Whole Genome CGH Tiling Set (2 of 8),NimbleGen Homo sapiens HG18 Whole Genome CGH Tiling Set (1 of 8),NimbleGen Homo sapiens HG18 Whole Genome CGH Tiling Set (3 of 8),NimbleGen Homo sapiens HG18 Whole Genome CGH Tiling Set (4 of 8),NimbleGen Homo sapiens HG18 Whole Genome CGH Tiling Set (7 of 8),NimbleGen Homo sapiens HG18 Whole Genome CGH Tiling Set (8 of 8),NimbleGen Homo sapiens HG18 Whole Genome CGH Tiling Set (5 of 8)','GRCh37,GRCh37.p13,GRCh38,GRCh38.p2,NCBI36','GCA_000001405.14,GCA_000001405.15,GCA_000001405.17,GCF_000001405.12,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd17','9606','Homo sapiens','Human','16327808','Conrad_et_al_2006','Control Set',null,'We report a new method that uses SNP genotype data from parent-offspring trios to identify polymorphic deletions. We applied this method to data from the International HapMap Project to produce the first high-resolution population surveys of deletion polymorphism. Approximately 100 of these deletions have been experimentally validated using comparative genome hybridization on tiling-resolution oligonucleotide microarrays. Our analysis identifies a total of 586 distinct regions that harbor deletion polymorphisms in one or more of the families.','Probe signal intensity,SNP genotyping analysis',null,'Oligo aCGH,SNP array','Sanger (Hurles) - Homo sapiens - 385K feature custom tiling path array - v1.0,Not reported','GRCh37,GRCh37.p13,GRCh38,GRCh38.p2,NCBI34','GCA_000001405.14,GCA_000001405.15,GCA_000001405.17,GCF_000001405.10,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd18','10090','Mus musculus','Mouse','17965714','Egan_et_al_2007','Control Set',null,'We used high-resolution microarrays to identify 39 CNVs among 14 colonies of the C57BL/6 strain spanning approximately 967 generations of inbreeding, and we examined these loci in 12 additional strains.','Merging,Manual observation,Probe signal intensity,Curated,Sequence alignment,Probe signal intensity,Probe signal intensity','HMM,HMM','Merging,PCR,ROMA,Curated,Sequencing,qPCR,Oligo aCGH','See merged experiments,Not reported,ROMA-mouse-85K-v1.0,Not reported,Not reported,Not reported,tiling-mouse-385K-v1.0','GRCm38,MGSCv36,MGSCv37','GCF_000001635.15,GCF_000001635.16,GCF_000001635.20');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd19','10090','Mus musculus','Mouse','20308636','Quinlan_et_al_2010','Control Set',null,'We used Illumina paired-end DNA sequencing to identify structural variation between two inbred mouse strains: DBA/2J and C57BL/6','Read depth,Probe signal intensity,Sequence alignment,Paired-end mapping',null,'Sequencing,qPCR,Sequencing,Sequencing','Solexa (Illumina GA2),Not reported,Not reported,Solexa (Illumina GA2)','GRCm38,GRCm38.p3,MGSCv37','GCF_000001635.16,GCF_000001635.20,GCF_000001635.23');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd2','9606','Homo sapiens','Human','18451855','Kidd_et_al_2008','Control Set','http://hgsv.washington.edu','We implemented an approach to construct clone-based maps of eight human genomes with the aim of systematically cloning and sequencing structural variants more than 8 kbp in length.','One end anchored assembly,Manual observation,MCD analysis,Paired-end mapping,Probe signal intensity,Probe signal intensity,Sequence alignment',null,'Sequencing,FISH,Multiple complete digestion,Sequencing,Oligo aCGH,Oligo aCGH,Sequencing','Not reported,Not reported,Not reported,Capillary,Not reported,Not reported,Not reported','GRCh37,GRCh37.p13,GRCh38,GRCh38.p2,NCBI35','GCA_000001405.14,GCA_000001405.15,GCA_000001405.17,GCF_000001405.11,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd20','9606','Homo sapiens','Human','16468122','McCarroll_et_al_2006','Control Set',null,'We developed a systematic approach for identifying deletions from patterns of Mendel failures, null genotypes and Hardy-Weinberg disequilibrium in dense genotype data.','SNP genotyping analysis,SNP genotyping analysis,SNP genotyping analysis',null,'SNP array,SNP array,SNP array','Not reported,Not reported,Not reported','GRCh37,GRCh37.p13,GRCh38,GRCh38.p2,NCBI34','GCA_000001405.14,GCA_000001405.15,GCA_000001405.17,GCF_000001405.10,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd21','9606','Homo sapiens','Human','19592680','Shaikh_et_al_2009','Control Set','http://cnv.chop.edu/','We present a database of copy number variations (CNVs) detected in 2,026 disease-free individuals, using high-density, SNP-based oligonucleotide microarrays. This large cohort analyzed for CNVs in a single study using a uniform array platform and computational tools, comprises mainly of Caucasians (65.2%) and African-Americans (34.2%), We have catalogued and characterized 54,462 individual CNVs, 77.8% of which were identified in multiple unrelated individuals. These non-unique CNVs mapped to 3,272 distinct regions of genomic variation spanning 5.9% of the genome; 51.5% of these were previously unreported, and &gt;85% are rare.','SNP genotyping analysis',null,'SNP array','GPL6434','GRCh37,GRCh37.p13,GRCh38,GRCh38.p2,NCBI35','GCA_000001405.14,GCA_000001405.15,GCA_000001405.17,GCF_000001405.11,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd22','9606','Homo sapiens','Human','18776908','McCarroll_et_al_2008','Control Set',null,'We sought to develop hybrid oligonucleotide microarrays to accurately analyze SNPs and copy number variation simultaneously; to use these arrays to map the genomic locations, allele frequencies and population-genetic properties of human CNPs; and to apply this knowledge to advance strategies for querying CNV in genome-wide association studies.','Probe signal intensity','Birdseye','SNP array','[GenomeWideSNP_6] Affymetrix Genome-Wide Human SNP 6.0 Array','GRCh37,GRCh37.p13,GRCh38,GRCh38.p2,NCBI35','GCA_000001405.14,GCA_000001405.15,GCA_000001405.17,GCF_000001405.11,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd23','9606','Homo sapiens','Human','18674749','Young_et_al_2008','Control Set',null,'We have undertaken a detailed study of copy-number variation of ORs to elucidate the selective and mechanistic forces acting on this gene family and the true impact of copy-number variation on human OR repertoires.','Manual observation,Probe signal intensity,Sequence alignment,Probe signal intensity,Sequence alignment','GeneMapper 3.7 or 4.0 software (Applied Biosystems)','PCR,MLPA,Sequencing,MLPA,Sequencing','Not reported,Not reported,Not reported,Not reported,Not reported','GRCh37,GRCh38,NCBI36','GCA_000001405.15,GCF_000001405.12,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd24','9823','Sus scrofa','Pig','19079605','Fadista_et_al_2008','Control Set',null,'Here we report a first account of CNVs in the pig genome covering part of the chromosomes 4, 7, 14, and 17 already sequenced and assembled. A custom tiling oligonucleotide array was used with a median probe spacing of 409 bp for screening 12 unrelated Duroc boars that are founders of a large family material. After a strict CNV calling pipeline, 37 copy number variable regions (CNVRs) across all four chromosomes were identified, with five CNVRs overlapping segmental duplications, three overlapping pig unigenes and one overlapping a RefSeq pig mRNA.','Probe signal intensity,Manual observation',null,'Oligo aCGH,RT-PCR','Nimblegen 385K pig array CGH,Not reported','Sscrofa10.2,Sscrofa3','GCF_000003025.5,Sscrofa3');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd25','7227','Drosophila melanogaster','Fruit fly','18056801','Dopman_et_al_2007','Control Set',null,'Here, we focus on copy-number variation in Drosophila melanogaster. We characterize copy-number polymorphism (CNP) across genomic regions, and we contrast patterns to infer the evolutionary processes acting on this variation.','Probe signal intensity',null,'Oligo aCGH','HartlLab Drosophilamelanogaster PCRproducts 22k','Release 5,Release 6 plus ISO1 MT','GCF_000001215.1,GCF_000001215.4');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd26','7227','Drosophila melanogaster','Fruit fly','18535209','Emerson_et_al_2008','Control Set',null,'We used high-density full-genome tiling arrays to create a fine-scale genomic map of copy-number polymorphisms (CNPs) in Drosophila melanogaster.','Sequence alignment,Probe signal intensity','HMM','Sequencing,Oligo aCGH','Not reported,Affymetrix Drosophila Genome Tiling Array 1.0F (Dm35b_MF_v02)','Release 4,Release 6 plus ISO1 MT','GCF_000001215.4,GCF_000004615.4');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd27','9606','Homo sapiens','Human','19166990','Itsara_et_al_2009','Control Set',null,'We identify large copy-number variants in ~2500 individuals by using Illumina SNP data, with an emphasis on â€˜â€˜hotspotsâ€™â€™ prone to recurrent mutations.','SNP genotyping analysis,Probe signal intensity',null,'SNP array,Oligo aCGH','Not reported,Not reported','GRCh37,GRCh37.p13,GRCh38,GRCh38.p2,NCBI35','GCA_000001405.14,GCA_000001405.15,GCA_000001405.17,GCF_000001405.11,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd28','9606','Homo sapiens','Human','19776401','Zhang_et_al_2009','Case-Set',null,'We employed a CGH based method to interrogate intermediate structural variation in 6 individuals of diverse geographic ancestry.','Probe signal intensity,Manual observation,Manual observation','Agilent Genome Workbench','Oligo aCGH,Western,PCR','GPL6264,Not reported,Not reported','GRCh37,GRCh38,NCBI36','GCA_000001405.15,GCF_000001405.12,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd29','9606','Homo sapiens','Human','16826518','Locke_et_al_2006','Control Set','http://humanparalogy.gs.washington.edu/structuralvariation/general/download.html','In an attempt to assess the heritability and LD of copy-number polymorphisms (CNPs) in duplication-rich regions of the genome, we profiled copy-number variation in 130 putative â€œrearrangement hotspot regionsâ€ among 269 individuals of European, Yoruba, Chinese, and Japanese ancestry analyzed by the International HapMap Consortium.','Probe signal intensity',null,'BAC aCGH','GPL4010','GRCh37,GRCh37.p13,GRCh38,GRCh38.p2,NCBI34','GCA_000001405.14,GCA_000001405.15,GCA_000001405.17,GCF_000001405.10,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd3','9544','Macaca mulatta','Rhesus monkey','18180252','Lee_et_al_2008','Control Set',null,'Performed a study of CNVs in the macaque genome by using a macaque-specific array-based comparative genomic hybridization (aCGH) platform to describe patterns of copy number variation among the genomes of 10 macaque individuals','Probe signal intensity','BreakPtr','Oligo aCGH','Nimblegen Macaque Whole genome CGH 385K array','Mmul_051212,Mmul_8.0.1','GCF_000002255.2,GCF_000772875.2');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd30','9606','Homo sapiens','Human','18288195','Jakobsson_et_al_2008','Control Set','http://neurogenetics.nia.nih.gov/paperdata/public/','Here we report the analysis of high-quality genotypes at 525,910 single-nucleotide polymorphisms (SNPs) and 396 copy-number-variable loci in a worldwide sample of 29 populations.','SNP genotyping analysis,SNP genotyping analysis','PennCNV,PennCNV','SNP array,SNP array','Illumina HumanHap550 Genotyping BeadChip v3,Illumina HumanHap550 Genotyping BeadChip v1','GRCh37,GRCh37.p13,GRCh38,GRCh38.p2,NCBI36','GCA_000001405.14,GCA_000001405.15,GCA_000001405.17,GCF_000001405.12,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd31','9606','Homo sapiens','Human','19718026','Alkan_et_al_2009','Control Set',null,'We present an algorithm (mrFAST) to comprehensively map next-generation sequence reads, which allows for the prediction of absolute copy-number variation of duplicated segments and genes. We examine three human genomes and experimentally validate genome-wide copy number differences. Our method provides a more accurate assessment of gene content and insight into functional constraint without the limitations of array-based technology.','Read depth,Probe signal intensity,Probe signal intensity,Read depth','mrFAST','Sequencing,Oligo aCGH,Oligo aCGH,Sequencing','Illumina Genome Analyzer and Roche/454 sequencer. For arrayCGH validation experiments we used Nimblegen,GPL13934,Not reported,Illumina Genome Analyzer and Roche/454 sequencer. For arrayCGH validation experiments we used Nimblegen','GRCh37,GRCh37.p13,GRCh38,GRCh38.p2,NCBI35','GCA_000001405.14,GCA_000001405.15,GCA_000001405.17,GCF_000001405.11,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd32','9606','Homo sapiens','Human','15918152','Sharp_et_al_2005','Control Set',null,'On the basis of the duplication architecture of the human genome, we defined a set of 130 potential rearrangement hotspots and constructed a targeted bacterial artificial chromosome (BAC) microarray (with 2,194 BACs) to assess copy-number variation in these regions by array comparative genomic hybridization. Using our segmental duplication BAC microarray, we screened a panel of 47 normal individuals, who represented populations from four continents, and we identified 119 regions of copy-number polymorphism (CNP), 73 of which were previously unreported.','Probe signal intensity,Manual observation',null,'BAC aCGH,FISH','GPL4010,Not reported','GRCh37,GRCh37.p13,GRCh38,GRCh38.p2,NCBI34','GCA_000001405.14,GCA_000001405.15,GCA_000001405.17,GCF_000001405.10,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd33','9606','Homo sapiens','Human','17360722','Sharp_et_al_2007','Case-Set','http://humanparalogy.gs.washington.edu/structuralvariation/','We describe multiple individuals with mental retardation and overlapping de novo submicroscopic deletions of 15q24 (1.7ï¿½ï¿½ï¿½3.9 Mb in size).','Probe signal intensity,Sequence alignment',null,'Oligo aCGH,Sequencing','Custom NimbeGen array,Not reported','GRCh37,GRCh38,NCBI35','GCA_000001405.15,GCF_000001405.11,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd34','9606','Homo sapiens','Human','16327809','Hinds_et_al_2006','Control Set',null,'We examined 100 deletion polymorphisms ranging from 70 bp to 7 kb.','Probe signal intensity,Manual observation',null,'Oligo aCGH,PCR','Custom Perlegen arrays ,Not reported','GRCh37,GRCh38,NCBI35','GCA_000001405.15,GCF_000001405.11,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd35','9606','Homo sapiens','Human','20440878','Kidd_et_al_2010','Control Set','http://hgsv.washington.edu/','Discovery and analysis of sequence insertions not represented in the human genome reference','One end anchored assembly,Sequence alignment,Sequence alignment,Probe signal intensity,Paired-end mapping,Paired-end mapping,One end anchored assembly','megaBLAST,megaBLAST','Sequencing,Sequencing,Sequencing,Oligo aCGH,Sequencing,Sequencing,Sequencing','Capillary,Agilent Custom Human 244K CGH Array,Capillary,Agilent Custom Human 244K CGH Array,Agilent Custom Human 244K CGH Array,Capillary,Agilent Custom Human 244K CGH Array','GRCh37,GRCh37.p13,GRCh38,GRCh38.p2,NCBI35,NCBI36','GCA_000001405.14,GCA_000001405.15,GCA_000001405.17,GCF_000001405.11,GCF_000001405.12,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd36','9606','Homo sapiens','Human','20598272','Mitsui_et_al_2010','Case-Set',null,'We performed a custom-designed high-density comparative genomic hybridization analysis to determine the junction sequences in germ cell lines and cancer cell lines involving PARK2 or DMD.','Probe signal intensity,Merging,Sequence alignment',null,'Oligo aCGH,Merging,Sequencing','Agilent,See merged experiments,Not reported','GRCh37,GRCh38,NCBI36','GCA_000001405.15,GCF_000001405.12,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd37','9606','Homo sapiens','Human','20466091','ClinGen_Laboratory-Submitted','Case-Set','http://www.clinicalgenome.org','Copy number variation identified through the course of routine clinical cytogenomic testing in postnatal populations, with clinical assertions as classified by the original submitter. For data from the original published study, [Kaminsky, et al. 2011|/pubmed/21844811], please see [nstd101|/dbvar/studies/nstd101/].','Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,SNP genotyping analysis,Probe signal intensity,Probe signal intensity,Manual observation,Probe signal intensity,Probe signal intensity,Manual observation,Probe signal intensity,SNP genotyping analysis,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Manual observation,Probe signal intensity,Probe signal intensity','Aberration algorithm = ADM-1 with 6.7 threshold; Aberration filter settings: minimum # probes = 4, minimum absolute average log ration = 0.26.,Aberration algorithm = ADM-1 with 6.7 threshold; Aberration filter settings: minimum # probes = 4, minimum absolute average log ration = 0.26.,Agilent: Aberration algorithm = ADM-1 with 6.7 threshold; Aberration filter settings: minimum # probes = 5, minimum absolute average log ration = 0.26.,Aberration algorithm = ADM-1 with 6.7 threshold; Aberration filter settings: minimum # probes = 5, minimum absolute average log ration = 0.26.,Agilent: Aberration algorithm = ADM-1 with 6.7 threshold; Aberration filter settings: minimum # probes = 5, minimum absolute average log ration = 0.26.','Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,MLPA,SNP array,Oligo aCGH,Oligo aCGH,FISH,Oligo aCGH,Oligo aCGH,PCR,Oligo aCGH,SNP array,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Karyotyping,Oligo aCGH,Oligo aCGH','ISCA 44K 024614, v2,EMArray Cyto6000,ISCA 8x60K OGT, v2,Agilent 023097,ISCA 4x180K, v2,OGT 60K 27574, v1,ISCA 44K 016267, v1,ISCA 8x60, v2,ISCA 60K 024612, v2,Affymetrix CytoScan HD,Human Genome CGH Microarray 44K, design ID 13282,Not reported,Not reported,Agilent ISCA 180K,Agilent 023097, v1,Not reported,ISCA 180K 024616, v2,ISCA 44K 16267, v1,Not reported,Not reported,Affymetrix CytoScan HD,Cytochip 180K, v1,Agilent custom 44K, v1,Not reported,OGT 60K 27574,ISCA 180k,SurePrint G3 Human CGH Microarray 4x180K (Agilent),Human Genome CGH Microarray 44K, design ID 14950,SurePrint G3 Human CGH Microarray, 8x60K,Not reported,ISCA 44K, v1,ISCA 8x60, v2,Agilent ISCA 44K,Bluegnome Cytochip 105K, v1.1,ISCA 180k,Cyto Custom HD-CGH Microarray (4x44K), design ID 16267,Not reported,Emory, 4x44K,Agilent ISCA 44K','GRCh37,GRCh37.p13,GRCh38,GRCh38.p2,NCBI36','GCA_000001405.14,GCA_000001405.15,GCA_000001405.17,GCF_000001405.12,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd39','9606','Homo sapiens','Human','20164927','Schuster_et_al_2010','Control Set',null,'Here we present the complete genome sequences of an indigenous hunter-gatherer from the Kalahari Desert and a Bantu from southern Africa, as well as protein-coding regions from an additional three hunter-gatherers from disparate regions of the Kalahari. We characterize the extent of whole-genome and exome diversity among the five men, reporting 1.3 million novel DNA differences genome-wide, including 13,146 novel amino acid variants.','Probe signal intensity,Read depth','mrFAST','Oligo aCGH,Sequencing','Not reported,Illumina Genome Analyzer and Roche/454 sequencer. For arrayCGH validation experiments we used Nimblegen','GRCh37,GRCh37.p13,GRCh38,GRCh38.p2,NCBI35','GCA_000001405.14,GCA_000001405.15,GCA_000001405.17,GCF_000001405.11,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd4','9606','Homo sapiens','Human','18304495','Perry_et_al_2008','Control Set','http://research.jax.org/faculty/charles-lee.html','We used a high-resolution array-based comparative genomic hybridization (aCGH) platform that targeted known CNV regions of the human genome at approximately 1 kb resolution to interrogate the genomic DNAs of 30 individuals from four HapMap populations','Probe signal intensity,Probe signal intensity','ADM2,ADM2','Oligo aCGH,Oligo aCGH','Agilent-015686 Custom Human 244K CGH Microarray,Agilent-015685 Custom Human 244K CGH Microarray','GRCh37,GRCh37.p13,GRCh38,GRCh38.p2,NCBI35','GCA_000001405.14,GCA_000001405.15,GCA_000001405.17,GCF_000001405.11,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd40','9606','Homo sapiens','Human','16906162','Sharp_et_al_2006','Case-Control','http://humanparalogy.gs.washington.edu/structuralvariation/','Based on the duplication architecture of the genome, we investigated 130 regions that we hypothesized as candidates for previously undescribed genomic disorders. We tested 290 individuals with mental retardation by BAC array comparative genomic hybridization and identified 16 pathogenic rearrangements, including de novo microdeletions of 17q21.31 found in four individuals.','Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity',null,'BAC aCGH,FISH,BAC aCGH,Oligo aCGH','GSE5373,Not reported,GSE5373,Custom NimbleGen array','GRCh37,GRCh37.p13,GRCh38,GRCh38.p2,NCBI34','GCA_000001405.14,GCA_000001405.15,GCA_000001405.17,GCF_000001405.10,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd41','9606','Homo sapiens','Human','15286789','Iafrate_et_al_2004','Control Set',null,'We identified 255 loci across the human genome that contain genomic imbalances among unrelated individuals. Data was originally reported on human genome assembly NCBI34; it is reported here on the current human genome assembly, GRCh37.','Probe signal intensity,Probe signal intensity',null,'BAC aCGH,FISH','Spectral Genomics 2600 BAC array,Not reported','GRCh37,GRCh38,GRCh38.p2','GCA_000001405.15,GCA_000001405.17,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd42','9606','Homo sapiens','Human','18278044','Sharp_et_al_2008','Case-Set',null,'We report a recurrent microdeletion syndrome causing mental retardation, epilepsy and variable facial and digital dysmorphisms.','Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity',null,'qPCR,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH,BAC aCGH','Not reported,University of Washington Human original custom 2-plex 166K array, version 1,University of Washington Human custom 2-plex 166K array for 15q13 region, A1 version 1,University of Washington Human custom 2-plex 166K array for 15q13 region, A2 subarray version 1,University of Washington Human Itsara PWS 388K custom design array, version 1,Not reported,Not reported','GRCh37,GRCh38,GRCh38.p2,NCBI35','GCA_000001405.15,GCA_000001405.17,GCF_000001405.11,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd43','9606','Homo sapiens','Human','19587683','Kim_et_al_2009','Control Set',null,'We detected structural variations using three methods: diploid GA sequencing, BAC end sequencing, and microarrays. 1. To confirm reliable large deletions in diploid GA sequencing, three criteria had to be met: (1) relative coverage drops compared to those of the flanking regions; (2) the existence of stretched paired-end sequence; and (3) the loss of heterozygous SNPs under the regions. Relative increase of sequencing coverage was used as a confirmation tool for copy number gains. 2. BACs less than 40 kb were considered as significantly short. The co-localization of two or more short BACs was considered as the candidate region of amplification. 3. Putative structural variations were detected with microarrays (Illumina BeadChip 370K, 610K, and Agilent 24M aCGH; Supplementary Methods). For Illumina BeadChips, normalized bead intensity data and genotype calls were obtained with Illumina BeadStudio 3.1 software. Results from Agilent 24M aCGH were analysed on Nexus software (BioDiscovery Inc.). Each aberration call was manually checked to confirm the accuracy of the calls.','Paired-end mapping,Probe signal intensity,Paired-end mapping,Probe signal intensity,Probe signal intensity','Nexus software,Illumina BeadStudio 3.1 software,Illumina BeadStudio 3.1 software','Sequencing,SNP array,Sequencing,SNP array,Oligo aCGH','GPL10329,GPL6985,GPL10329,GPL8887,GSE19651','GRCh37,GRCh37.p13,GRCh38,GRCh38.p2,NCBI36','GCA_000001405.14,GCA_000001405.15,GCA_000001405.17,GCF_000001405.12,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd44','9823','Sus scrofa','Pig','20969757','Ramayo-Caldas_et_al_2010','Control Set',null,'We used predictions from three different programs (cnvPartition , PennCNV and GADA) to analyze data from the Porcine SNP60 BeadChip. A total of 49 CNVRs were identified in 55 animals from an Iberian x Landrace cross (IBMAP) according to three criteria: detected in at least two animals, contained three or more consecutive SNPs and recalled by at least two programs. Mendelian inheritance of CNVRs was confirmed in animals belonging to several generations of the IBMAP cross. Subsequently, a segregation analysis of these CNVRs was performed in 372 additional animals from the IBMAP cross and its distribution was studied in 133 unrelated pig samples from different geographical origins. Five out of seven analyzed CNVRs were validated by real time quantitative PCR.','SNP genotyping analysis,Probe signal intensity',null,'SNP array,qPCR','Not reported,Not reported','Sscrofa10.2,Sscrofa9','GCA_000003025.1,GCF_000003025.5');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd45','9606','Homo sapiens','Human','22097934','ClinGen_Curated_Dosage_Sensitivity_Map','Collection','http://www.clinicalgenome.org','Genes/genomic regions with sufficient evidence supporting (pathogenic) or refuting (benign) dosage sensitivity as a mechanism for disease. Evidence is evaluated on a continual basis by the ClinGen Structural Variation Working Group as described in Riggs et al. 2012.','Curated',null,'Curated','Not reported','GRCh37,GRCh38,GRCh38.p2,NCBI36','GCA_000001405.15,GCA_000001405.17,GCF_000001405.12,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd46','9606','Homo sapiens','Human','21397061','Campbell_et_al_2011','Control Set',null,'Copy number variants (CNVs) can reach appreciable frequencies in the human population, and several of these copy number polymorphisms (CNPs) have been recently associated with human diseases including lupus, psoriasis, Crohn disease, and obesity. Despite new advances, significant biases remain in terms of CNP discovery and genotyping. We developed a novel method based on single channel intensity data and benchmarked against copy numbers determined from sequencing read-depth to successfully obtain CNP genotypes for 1489 CNPs from 487 human DNA samples from diverse ethnic backgrounds. This customized microarray was enriched for segmental duplication-rich regions and novel insertions of sequences not represented in the reference genome assembly or on standard single nucleotide polymorphism (SNP) microarray platforms. We observe that CNPs in segmental duplications are more likely to be population differentiated than CNPs in unique regions (p = 0.015) and that bi-allelic CNPs show greater stratification when compared to frequency-matched SNPs (p = 0.0026). Although bi-allelic CNPs show a strong correlation of copy number with flanking SNP genotypes, the majority of multi-copy CNPs do not (40% with r &gt;0.8). We selected a subset of CNPs for further characterization in 1873 additional samples from 62 populations; this revealed striking population-differentiated structural variants in genes of clinical significance such as the OCLN gene, a tight junction protein involved in hepatitis C viral entry. Our new microarray design allows these variants to be rapidly tested for disease association and our results suggest that CNPs (especially those not in linkage disequilibrium with SNPs) may have contributed disproportionately to human diversity and selection.','Probe signal intensity,Probe signal intensity','Agilent Genomic Workbench ADM2 algorithm,Agilent Genomic Workbench ADM2 algorithm','Oligo aCGH,Oligo aCGH','Agilent Eichler Human CNP 180K v2.1,Agilent Eichler Human CNP 180K v3.0','GRCh37,GRCh37.p13,GRCh38,GRCh38.p2,NCBI36','GCA_000001405.14,GCA_000001405.15,GCA_000001405.17,GCF_000001405.12,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd47','9606','Homo sapiens','Human','21111241','Kidd_et_al_2010b','Control Set','http://hgsv.washington.edu/','A human genome structural variation sequencing resource reveals insights into mutational mechanisms','Sequence alignment',null,'Sequencing','Capillary','GRCh37,GRCh37.p13,GRCh38,GRCh38.p2,NCBI36','GCA_000001405.14,GCA_000001405.15,GCA_000001405.17,GCF_000001405.12,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd48','10090','Mus musculus','Mouse','19270705','Henrichsen_et_al_2009','Control Set',null,'We identified autosomal copy number variants in inbred mouse strains and field mice, using oligonucleotide-based array comparative genome hybridisation.','Probe signal intensity,Probe signal intensity,Probe signal intensity,Probe signal intensity',null,'Oligo aCGH,Oligo aCGH,Oligo aCGH,Oligo aCGH','GPL11047,MM6 385K Whole Genome Tiling array,GPL10992,Custom CGH array tiling selected regions on MM6 [2007-04-16_MM6_GK_CGH]','GRCm38,GRCm38.p3,MGSCv34,MGSCv37','GCF_000001635.13,GCF_000001635.16,GCF_000001635.20,GCF_000001635.23');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd49','9606','Homo sapiens','Human','20534489','Teague_et_al_2010','Control Set','http://www.lmcg.wisc.edu','We analyzed four phenotypically normal human genomes using Optical Mapping, a single-molecule platform for genome structure analysis.','Paired-end mapping,Optical mapping,Probe signal intensity,Probe signal intensity,Probe signal intensity,Paired-end mapping,Probe signal intensity,Probe signal intensity,One end anchored assembly,Probe signal intensity','SOMA, Gentig, iterative assembly','Sequencing,Optical mapping,Oligo aCGH,Oligo aCGH,Oligo aCGH,Sequencing,Oligo aCGH,Oligo aCGH,Sequencing,BAC aCGH','Not reported,Optical Mapping,Not reported,[Mapping250K_Nsp] Affymetrix Mapping 250K Nsp SNP Array,[Mapping250K_Sty] Affymetrix Mapping 250K Sty2 SNP Array,Not reported,Not reported,Not reported,Not reported,Not reported','GRCh37,GRCh37.p13,GRCh38,GRCh38.p2,NCBI35','GCA_000001405.14,GCA_000001405.15,GCA_000001405.17,GCF_000001405.11,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd5','10090','Mus musculus','Mouse','17206864','Graubert_et_al_2007','Control Set',null,'We chose to study 21 well-characterized inbred mouse strains that are the focus of an international effort to measure, catalog, and disseminate phenotype data. We performed comparative genomic hybridization using long oligomer arrays to characterize CNVs in these strains.','Probe signal intensity','Bioconductor','Oligo aCGH','NimbleGen Mouse 388K NCBI34','GRCm38,GRCm38.p3,MGSCv34,MGSCv37','GCF_000001635.13,GCF_000001635.16,GCF_000001635.20,GCF_000001635.23');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd50','9606','Homo sapiens','Human','21212237','Arlt_et_al_2011','Control Set',null,'To compare available whole-genome approaches for scoring genome structural variation, data from both 1M feature single-nucleotide polymorphism (SNP) arrays and low-density mate-pair sequencing were mined using a new computational platform, VAMP, to detect both constitutional CNVs in a single individual as well as de novo CNVs induced by the replication stressor aphidicolin.','Paired-end mapping,SNP genotyping analysis,SNP genotyping analysis','VAMP, http://tewlab.path.med.umich.edu/vamp/doc/vampDoc.html,VAMP, http://tewlab.path.med.umich.edu/vamp/doc/vampDoc.html,VAMP, http://tewlab.path.med.umich.edu/vamp/doc/vampDoc.html','Sequencing,SNP array,SNP array','Not reported,GPL8882,Illumina HumanOmni1-Quad BeadChip','GRCh37,GRCh37.p13,GRCh38,GRCh38.p2,NCBI36','GCA_000001405.14,GCA_000001405.15,GCA_000001405.17,GCF_000001405.12,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd51','9606','Homo sapiens','Human',null,'User_submitted_curated_variants','Collection',null,'User submitted curated variants from OMIM, GeneReviews, or ClinVar','Merging,Curated,Curated,Curated','Coordinates for structural variants may be as reported in source or calculated - from features (e.g., affected genes or flanking probes), HGVS name, or cytoband.,Coordinates for structural variants may be as reported in source or calculated - from features (e.g., affected genes or flanking probes), HGVS name, or cytoband.,Coordinates for structural variants may be as reported in source or calculated - from features (e.g., affected genes or flanking probes), HGVS name, or cytoband.','Merging,Curated,Curated,Curated','Not reported,Not reported,Not reported,Not reported','GRCh37,GRCh37.p13,GRCh38,GRCh38.p2,NCBI35,NCBI36','GCA_000001405.14,GCA_000001405.15,GCA_000001405.17,GCF_000001405.11,GCF_000001405.12,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd52','9606','Homo sapiens','Human',null,'Watkins_et_al_2011','Case-Set',null,'The study identified a large structural variant in an unaffected parent that was transmitted to a stillborn proband with infantile arterial calcification.','SNP genotyping analysis,Probe signal intensity',null,'SNP array,qPCR','[GenomeWideSNP_6] Affymetrix Genome-Wide Human SNP 6.0 Array,Not reported','GRCh37,GRCh38,NCBI36','GCA_000001405.15,GCF_000001405.12,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd53','9606','Homo sapiens','Human','21559371','Tuttelmann_et_al_2011','Case-Control',null,'Comparison of 89 infertile male patients with severe oligozoospermia (=5 x Mill./ml sperm concentration and =10 Mill. total sperm count) and 37 with azoospermia due to complete, bilateral Sertoli-cell-only syndrome (SCOS) with 100 healthy controls with normal semen parameters (=20 Mill./ml sperm concentration, =40 Mill. total sperm count, =2 ml semen volume, =50% of grade a+b or =25% grade a motility, high percentage of normal forms (=10%)). Patient-specific and either recurring or private, sex-chromosomal CNVs are reported as possibly causing spermatogenic failure.','Probe signal intensity,Probe signal intensity','Analysis with ''DNA Analyticsï¿½ï¿½ï¿½ (version 4.0.81, Agilent Technologies),Analysis with ''DNA Analyticsï¿½ï¿½ï¿½ (version 4.0.81, Agilent Technologies)','Oligo aCGH,Oligo aCGH','Agilent-021850 SurePrint G3 Human CGH Microarray 2x400K,Agilent-014693 Human Genome CGH Microarray 244A (Feature number version)','GRCh37,GRCh37.p13,GRCh38,GRCh38.p2,NCBI36','GCA_000001405.14,GCA_000001405.15,GCA_000001405.17,GCF_000001405.12,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd54','9606','Homo sapiens','Human','21841781','Cooper_et_al_2011','Case-Control',null,'Copy Number Variants from 15,767 cases of Developmental Delay and Intellectual Disability from Signature Genomics, and 8329 Control Samples. This study contains samples in common with [Coe et al. 2014|/dbvar/studies/nstd100]. Due to analysis differences (see manuscripts) please use the case samples (Sampleset 1) from only one of these submissions. Control sample sets do not overlap and may be combined.','SNP genotyping analysis,Probe signal intensity','A Hidden Markov Model (HMM) based on both allele frequencies and total intensity values (details in PMID: 19166990, 17384021) was used to identify putative alterations, followed by manual inspection of large CNVs (&gt;100 probes and &gt;1 Mbp) in conjunction with user guided merging of nearby (&lt;1 Mbp between for arrays with &lt;1 million probes and &lt;200 kbp for arrays with &gt;1 million probes) calls, which represent a single region broken up by the HMM, or gaps.','SNP array,Oligo aCGH','Not reported,Not reported','GRCh37,GRCh37.p13,GRCh38,GRCh38.p2,NCBI36','GCA_000001405.14,GCA_000001405.15,GCA_000001405.17,GCF_000001405.12,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd55','9606','Homo sapiens','Human','21636067','Zhu_et_al_2011','Control Set',null,'We used a PCR-based sequencing method to detect deletions mediated by a human-specific palindromic sequence in 740 individuals of different ethnic origins.','Sequence alignment',null,'Sequencing','Not reported','GRCh37,GRCh38','GCA_000001405.15,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd56','9913','Bos taurus','Cow','20212021','Liu_et_al_2010','Control Set',null,'We describe the first systematic and genome-wide analysis of copy number variations (CNVs) in modern domesticated cattle using array CGH, qPCR and FISH for 90 animals from 11 Bos taurus, 3 Bos indicus and 3 composite breeds for beef, dairy or dual purpose.','Probe signal intensity,Probe signal intensity,Probe signal intensity',null,'qPCR,FISH,Oligo aCGH','Not reported,Not reported,Not reported','Bos_taurus_UMD_3.1.1,Btau_4.0,Btau_5.0.1','GCF_000003055.5,GCF_000003205.2,GCF_000003205.7');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd57','9796','Equus caballus','Horse','22340285','Doan_et_al_2012','Control Set',null,'Whole genome sequencing of Quarter Horse','Read depth,Sequence alignment,Probe signal intensity','FREEC Software,CLC Genomics,ADAM-2 Algorithm','Sequencing,Sequencing,Oligo aCGH','Illumina GA II,Illumina GA II,aCGH Exon array','EquCab2.0','GCF_000002305.2');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd58','9606','Homo sapiens','Human','22305530','Forsberg_et_al_2012','Control Set',null,'Using age-stratified cohorts of monozygotic twins and singleborn subjects, we describe age-related accumulation of copy number variation (CNV) in the nuclear genomes in vivo. DNA from peripheral blood was run on Illumina genotyping arrays and the LogR Ratio and B-allele-frequency data were used to identify somatic copy number events.','SNP genotyping analysis,SNP genotyping analysis,SNP genotyping analysis,SNP genotyping analysis,SNP genotyping analysis,SNP genotyping analysis',null,'SNP array,SNP array,SNP array,SNP array,SNP array,SNP array','Illumina Human660W Quad v1,Illumina Human1M-Duov3 DNA Analysis BeadChip (Human1M-Duov3_B),Illumina Human1M Duo v3,Illumina Human660W-Quad v1.0 BeadChip,Illumina Human1M-Duov3 DNA Analysis BeadChip (Human1M-Duov3_B),Illumina Human660W-Quad v1.0 BeadChip','GRCh37,GRCh38,NCBI36','GCA_000001405.15,GCF_000001405.12,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd59','9606','Homo sapiens','Human','22608502','Lopez-Herrera_et_al_2012','Case-Set',null,'Deleterious mutations in LRBA are associated with a novel syndrome of immune deficiency and autoimmunity','Sequence alignment',null,'Sequencing','Not reported','GRCh37,GRCh38,NCBI36','GCA_000001405.15,GCF_000001405.12,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd6','9606','Homo sapiens','Human','16902084','Mills_et_al_2006','Control Set',null,'In this report, we describe an initial map of human INDEL variation that contains 415,434 unique INDEL polymorphisms. These INDELs were identified with a computational approach using     DNA re-sequencing traces originally generated for SNP discovery.','Sequence alignment',null,'Sequencing','Not reported','GRCh37,GRCh37.p13,GRCh38,GRCh38.p2,NCBI35','GCA_000001405.14,GCA_000001405.15,GCA_000001405.17,GCF_000001405.11,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd60','9913','Bos taurus','Cow','21345189','Hou_et_al_2011','Control Set',null,'We performed a systematic analysis of CNV using the Bovine HapMap SNP genotyping data, including 521 animals of 21 modern cattle breeds. After correcting genomic waves and considering the pedigree information, we identified 682 candidate CNV regions, which represent 139.8 megabases (~4.60%) of the genome. Selected CNVs were further experimentally validated and we found that copy number "gain" CNVs were predominantly clustered in tandem rather than existing as interspersed duplications. Many CNV regions (~56%) overlap with cattle genes (1,263), which are significantly enriched for immunity, lactation, reproduction and rumination. The overlap of this new dataset and other published CNV studies was less than 40%; however, our discovery of large, high frequency (&gt; 5% of animals surveyed) CNV regions showed 90% agreement with other studies. These results highlight the differences and commonalities between technical platforms.','SNP genotyping analysis,Probe signal intensity','PennCNV','SNP array,qPCR','BovineSNP50,Not reported','Bos_taurus_UMD_3.1.1,Btau_4.0,Btau_5.0.1','GCF_000003055.5,GCF_000003205.2,GCF_000003205.7');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd61','9913','Bos taurus','Cow','21928070','Hou_et_al_2011b','Case-Control',null,'In this study, we performed a large-scale analysis of CNVs using SNP genotyping data from 472 Angus cattle that were segregated for parasite resistance and susceptibility to gastrointestinal nematodes. To investigate the functional impacts of CNVs, we created 2 groups of 100 individual animals with extremely low or high estimated breeding values of eggs per gram of feces and referred to these groups as parasite resistant (PR) or parasite susceptible (PS), respectively. We identified 297 (~51 Mb) and 282 (~48 Mb) CNV regions from PR and PS groups, respectively. Approximately 60% of the CNV regions were specific to the PS group or PR group of animals. Selected PR- or PS-specific CNVs were further experimentally validated by quantitative PCR. A total of 297 PR CNV regions overlapped with 437 Ensembl genes enriched in immunity and defense, like WC1 gene which uniquely expresses on gamma/delta T cells in cattle. Network analyses indicated that the PR-specific genes were predominantly involved in gastrointestinal disease, immunological disease, inflammatory response, cell-to-cell signaling and interaction, lymphoid tissue development, and cell death. By contrast, the 282 PS CNV regions contained 473 Ensembl genes which are overrepresented in environmental interactions. Network analyses indicated that the PS-specific genes were particularly enriched for inflammatory response, immune cell trafficking, metabolic disease, cell cycle, and cellular organization and movement.','SNP genotyping analysis,Probe signal intensity','PennCNV','SNP array,qPCR','BovineSNP50,Not reported','Bos_taurus_UMD_3.1.1,Btau_4.0,Btau_5.0.1','GCF_000003055.5,GCF_000003205.2,GCF_000003205.7');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd62','7955','Danio rerio','Zebrafish','22203992','Brown_et_al_2012','Control Set',null,'Comparative genomic hybridization analysis of 3 laboratory and one wild zebrafish populations for Copy Number Variants','Probe signal intensity','Nexus copy number software','Oligo aCGH','Agilent','GRCz10,Zv8,Zv9','GCF_000002035.3,GCF_000002035.4,GCF_000002035.5');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd63','4558','Sorghum bicolor','Sorghum','22104744','Zheng_et_al_2011','Control Set','http://gigadb.org/dataset/100012','Sorghum (Sorghum bicolor) is globally produced as a source of food, feed, fibre and fuel. Grain and sweet sorghums differ in a number of important traits including stem sugar and juice accumulation, plant height and production of grain and biomass. The first whole genome sequence of a grain sorghum is available, but additional genome sequences are required to study genome-wide and intraspecies variation for dissecting the genetic basis of these important traits and for tailor-designed breeding of this important C4 crop. We resequenced two sweet and one grain sorghum inbred lines, and identified a set of nearly 1,500 genes differentiating sweet and grain sorghum. In addition, we uncovered 1,057,018 SNPs, 99,948 indels of 1-10bp in length and 16,487 presence/absence variations. In addition, 17,111 CNVs were detected. This is a first report on the identification of genome-wide patterns of genetic variation in sorghum.','Read depth,Paired-end mapping','soapCNV,soapSV','Sequencing,Sequencing','Illumina Solexa Genome Analyzer,Illumina Solexa Genome Analyzer','Sorbi1','Sorbi1');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd64','9606','Homo sapiens','Human','17921354','Wang_et_al_2007','Control Set',null,'Comprehensive identification and cataloging of copy number variations (CNVs) is required to provide a complete view of human genetic variation. The resolution of CNV detection in previous experimental designs has been limited to tens or hundreds of kilobases. Here we present PennCNV, a hidden Markov model (HMM) based approach, for kilobase-resolution detection of CNVs from Illumina high-density SNP genotyping data. This algorithm incorporates multiple sources of information, including total signal intensity and allelic intensity ratio at each SNP marker, the distance between neighboring SNPs, the allele frequency of SNPs, and the pedigree information where available. We applied PennCNV to genotyping data generated for 112 HapMap individuals; on average, we detected approximately 27 CNVs for each individual with a median size of approximately 12 kb. Excluding common rearrangements in lymphoblastoid cell lines, the fraction of CNVs in offspring not detected in parents (CNV-NDPs) was 3.3%. Our results demonstrate the feasibility of whole-genome fine-mapping of CNVs via high-density SNP genotyping.','Probe signal intensity','PennCNV','SNP array','GPL6433','GRCh37,GRCh37.p13,GRCh38,GRCh38.p2,NCBI35','GCA_000001405.14,GCA_000001405.15,GCA_000001405.17,GCF_000001405.11,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd65','9606','Homo sapiens','Human','20802225','Ju_et_al_2010','Control Set',null,'Comparative genomic hybridization (CGH) microarrays have been used to determine copy number variations (CNVs) and their effects on complex diseases. Detection of absolute CNVs independent of genomic variants of an arbitrary reference sample has been a critical issue in CGH array experiments. Whole genome analysis using massively parallel sequencing with multiple ultra-high resolution CGH arrays provides an opportunity to catalog highly accurate genomic variants of the reference DNA (NA10851). Using information on variants, we developed a new method, the CGH array referencefree algorithm (CARA), which can determine reference-unbiased absolute CNVs from any CGH array platform. The algorithm enables the removal and rescue of false positive and false negative CNVs, respectively, which appear due to the effects of genomic variants of the reference sample in raw CGH array experiments. We found that the CARA remarkably enhanced the accuracy of CGH array in determining absolute CNVs. Our method thus provides a new approach to interpret CGH array data for personalized medicine.','Read depth','CGH array reference free algorithm (CARA)','Sequencing','Illumina IIx','GRCh37,GRCh37.p13,GRCh38,GRCh38.p2,NCBI36','GCA_000001405.14,GCA_000001405.15,GCA_000001405.17,GCF_000001405.12,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd66','9606','Homo sapiens','Human','15273396','Sebat_et_al_2004','Control Set',null,'The extent to which large duplications and deletions contribute to human genetic variation and diversity is unknown. Here, we show that large-scale copy number polymorphisms (CNPs) - about 100 kilobases and greater - contribute substantially to genomic variation between normal humans. Representational oligonucleotide microarray analysis of 20 individuals revealed a total of 221 copy number differences representing 76 unique CNPs. On average, individuals differed by 11 CNPs, and the average length of a CNP interval was 465 kilobases. We observed copy number variation of 70 different genes within CNP intervals, including genes involved in neurological function, regulation of cell growth, regulation of metabolism, and several genes known to be associated with disease.','Probe signal intensity','S-PLUS Software','ROMA','A custom Nimblegen photoprint 85,000 probe ROMA oligo array platform was used.','GRCh37,GRCh37.p13,GRCh38,GRCh38.p2,NCBI35','GCA_000001405.14,GCA_000001405.15,GCA_000001405.17,GCF_000001405.11,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd67','9606','Homo sapiens','Human','20364138','Park_et_al_2010','Control Set',null,'Copy number variants (CNVs) account for the majority of human genomic diversity in terms of base coverage. Here, we have developed and applied a new method to combine high-resolution array comparative genomic hybridization (CGH) data with whole-genome DNA sequencing data to obtain a comprehensive catalog of common CNVs in Asian individuals. The genomes of 30 individuals from three Asian populations (Korean, Chinese and Japanese) were interrogated with an ultra-high-resolution array CGH platform containing 24 million probes. Whole-genome sequencing data from a reference genome (NA10851, with 28.3ï¿½ï¿½ coverage) and two Asian genomes (AK1, with 27.8ï¿½ï¿½ coverage and AK2, with 32.0ï¿½ï¿½ coverage) were used to transform the relative copy number information obtained from array CGH experiments into absolute copy number values. We discovered 5,177 CNVs, of which 3,547 were putative Asian-specific CNVs. These common CNVs in Asian populations will be a useful resource for subsequent genetic studies in these populations, and the new method of calling absolute CNVs will be essential for applying CNV data to personalized medicine.','Probe signal intensity','The ADM2 statistical algorithm was used to identify CNVs based on the combined log2ratios.','Oligo aCGH','Agilent 24M aCGH','GRCh37,GRCh37.p13,GRCh38,GRCh38.p2,NCBI36','GCA_000001405.14,GCA_000001405.15,GCA_000001405.17,GCF_000001405.12,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd68','9606','Homo sapiens','Human','17160897','Wong_et_al_2007','Control Set',null,'Segmental copy-number variations (CNVs) in the human genome are associated with developmental disorders and susceptibility to diseases. More importantly, CNVs may represent a major genetic component of our phenotypic diversity. In this study, using a whole-genome array comparative genomic hybridization assay, we identified 3,654 autosomal segmental CNVs, 800 of which appeared at a frequency of at least 3%. Of these frequent CNVs, 77% are novel. In the 95 individuals analyzed, the two most diverse genomes differed by at least 9 Mb in size or varied by at least 266 loci in content. Approximately 68% of the 800 polymorphic regions overlap with genes, which may reflect human diversity in senses (smell, hearing, taste, and sight), rhesus phenotype, metabolism, and disease susceptibility. Intriguingly, 14 polymorphic regions harbor 21 of the known human microRNAs, raising the possibility of the contribution of microRNAs to phenotypic diversity in humans. This in-depth survey of CNVs across the human genome provides a valuable baseline for studies involving human genetics.','Probe signal intensity',null,'BAC aCGH','GPL2616','GRCh37,GRCh37.p13,GRCh38,GRCh38.p2,NCBI35','GCA_000001405.14,GCA_000001405.15,GCA_000001405.17,GCF_000001405.11,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd69','9913','Bos taurus','Cow','22300768','Bickhart_et_al_2012','Control Set',null,'Using a read depth approach based on next-generation sequencing, we examined genome-wide copy number differences among five taurine (three Angus, one Holstein and one Hereford) and one indicine (Nelore) cattle. We identified 1,265 CNV regions comprising ~55.6 Mbp sequence-476 of which (~38%) have not previously been reported. Genes related to pathogen- and parasite-resistance, such as CATHL4 and ULBP17, were highly duplicated in the Nelore individual relative to the taurine cattle, while genes involved in lipid transport and metabolism, including APOL3 and FABP2, were highly duplicated in the beef breeds. These CNV regions also harbor genes like BPIFA2A (BSP30A) and WC1, suggesting that some CNVs may be associated with breed-specific differences in adaptation, health, and production traits.','Probe signal intensity,Read depth,Probe signal intensity,Read depth',null,'FISH,Sequencing,qPCR,Sequencing','Not reported,Sanger,BioRad iCycler,Illumina GAIIx','Bos_taurus_UMD_3.1.1,Btau_4.0,Btau_5.0.1','GCF_000003055.5,GCF_000003205.2,GCF_000003205.7');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd7','10090','Mus musculus','Mouse','19270704','Cahan_et_al_2009','Control Set','http://graubertlab.dom.wustl.edu/downloads.html','We analyzed the copy number content of the mouse genome to subï¿½ï¿½ï¿½10-kb resolution. We identified over 1,300 copy number variant regions (CNVRs), most of which are < 10 kb in length, are found in more than one strain, and, in total, span 3.2% (85 Mb) of the genome.','Probe signal intensity,Probe signal intensity','wuHMM,wuHMM','Oligo aCGH,Oligo aCGH','NimbleGen Mouse 2.1M NCBI36,WU Mouse_Illumina_46k_v6-1.1','GRCm38,GRCm38.p3,MGSCv36,MGSCv37','GCF_000001635.15,GCF_000001635.16,GCF_000001635.20,GCF_000001635.23');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd70','10090','Mus musculus','Mouse','23110596','Yasuda_et_al_2012','Control Set',null,'To evaluate the accuracy of computational tools that detect large deletions based on next generation sequencing, we generated deletion calls based on Sanger reads. This data contains deletions in chromosome 1 of the DBA/2J strain against mm9. The coordinates are in base-pair level resolution. We generated this data in following steps: (1) Sanger reads were obtained from the NCBI trace archive database. (2) These sequences were mapped to chromosome 1 of mm9 by MegaBLAST. (3) Our custom perl script found split Sanger reads that indicate deletions. (4) Deletions indicated by Sanger reads were merged if their coordinates were within 5 bases.','Split read mapping','megablast','Sequencing','Not reported','GRCm38,GRCm38.p3,MGSCv37','GCF_000001635.16,GCF_000001635.20,GCF_000001635.23');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd71','9606','Homo sapiens','Human','21882294','Xu_et_al_2011','Control Set',null,'Singapore Database of Copy Number Variants (SgD-CNV)','SNP genotyping analysis,SNP genotyping analysis','PennCNV,PennCNV','SNP array,SNP array','Illumnina HumanHap 610 Quad,Illumnina HumanHap 1MduoV3','GRCh37,GRCh37.p13,GRCh38,GRCh38.p2,NCBI36','GCA_000001405.14,GCA_000001405.15,GCA_000001405.17,GCF_000001405.12,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd72','9606','Homo sapiens','Human','23091296','Morak_et_al_2012','Case-Set',null,'We have discovered two new intrachromosomal breakpoints in the region upstream of CRLF2 exon1, which are involved in P2RY8-CRLF2 rearrangements in childhood ALL. This breakpoints are approximately 0.1 kb and 2.1 kb distal to CRLF2 exon1.','Sequence alignment',null,'Sequencing','Not reported','GRCh37,GRCh38','GCA_000001405.15,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd73','9606','Homo sapiens','Human','24416366','Dogan_et_al_2014','Control Set',null,'In this study we have sequenced the whole genome of an anonymous healthy male Turkish individual with high coverage (~35x). Resulting high quality data represented ~1.18 billion paired-end reads accounting for ~116,720 M bp. The structural variations (SV) submitted in this entry have been identified using paired-end and read-depth based SV calling algorithms.','Paired-end mapping,Read depth','CLEVER,CNVnator','Sequencing,Sequencing','Illumina HiSeq 2000,Illumina HiSeq 2000','GRCh37,GRCh38,GRCh38.p2','GCA_000001405.15,GCA_000001405.17,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd74','10090','Mus musculus','Mouse','23321323','Kazmi_et_al_2013','Tumor vs. Matched-Normal',null,'Array CGH analysis of malignant peripheral nerve sheath tumors (MPNSTs) arising in neuregulin-1 overexpressing (P0-GGFbeta3) mice','Probe signal intensity','â€˜Feature Extraction v9.5â€™ and â€˜DNA Analytics v4.0â€™ software (Agilent Technologies)','Oligo aCGH','Agilent','GRCm38,GRCm38.p3,MGSCv37','GCF_000001635.16,GCF_000001635.20,GCF_000001635.23');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd75','9606','Homo sapiens','Human','23215555','ClinGen_Prenatal','Case-Set','http://www.clinicalgenome.org','Prenatal Dataset, ClinGen. This study contains the microarray data from a large, prospective study of prenatal diagnostic samples to assess, in blinded fashion, the ability of microarray analysis to diagnose common chromosome abnormalities and to gauge the extent of additional information provided by microarray analysis as compared with standard karyotyping.','Probe signal intensity,Manual observation,Manual observation,Manual observation',null,'Oligo aCGH,FISH,Karyotyping,PCR','Not reported,Not reported,Not reported,Not reported','GRCh37,GRCh37.p13,GRCh38,GRCh38.p2,NCBI36','GCA_000001405.14,GCA_000001405.15,GCA_000001405.17,GCF_000001405.12,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd76','9606','Homo sapiens','Human','23541343','Watson_et_al_2013','Control Set',null,'The immunoglobulin heavy chain locus (IGH) encodes variable (IGHV), diversity (IGHD), joining (IGHJ), and constant (IGHC) genes, and is responsible for antibody heavy chain biosynthesis. Using a hydatidiform mole BAC clone resource, and fosmid clone libraries from nine diploid genomes of diverse ethnic origin, we have characterized 9 large structural variants (4 insertions; 1 tandem duplication; 2 deletions; 2 complex events) in this region at nucleotide resolution, accounting for over 220 Kbp of novel sequence.','Sequence alignment',null,'Sequencing','capillary-based Sanger sequencing','GRCh37,GRCh38','GCA_000001405.15,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd77','9606','Homo sapiens','Human','26813944','Polley_et_al_2016','Case-Control',null,'DMBT1 has a complex copy number variable structure, with two, independent, rapidly mutating copy number variable regions, called CNV1 and CNV2. Because the copy number variable regions are predicted to affect the number of bacteria-binding domains, different alleles may alter host-microbe interactions in the gut. Our aim was to investigate the role of this complex variation in susceptibility to Crohn''s disease. We analysed the association of both copy number variable regions with presence of Crohn''s disease, and its severity, on three case-control cohorts. We also reanalysed array comparative genomic hybridisation data (aCGH) from a large case-control cohort study for both copy number variable regions. We found no association with a linear increase in copy number, nor when the CNV1 is regarded as presence or absence of a deletion allele.','Probe signal intensity,Probe signal intensity,Probe signal intensity','CNVtools,CNVtools','Oligo aCGH,qPCR,qPCR','Not reported,Not reported,Not reported','GRCh37,GRCh38','GCA_000001405.15,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd78','9606','Homo sapiens','Human','23359205','Schrider_et_al_2013','Control Set',null,'An important class of gene copy-number polymorphism is gene duplications caused by retrotransposition, which result in a new intron-less copy of the parental gene (retroCNV) being inserted into a random location in the genome. Here we report the first genome-wide analysis of these variants in humans. We find that retroCNVs account for a substantial fraction of gene copy-number differences between any two individuals. Moreover, we show that these variants may often result in expressed chimeric transcripts, underscoring their potential influence in the evolution of novel gene functions.','Paired-end mapping,Manual observation',null,'Sequencing,PCR','Not reported,Not reported','GRCh37,GRCh38,GRCh38.p2','GCA_000001405.15,GCA_000001405.17,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd79','9606','Homo sapiens','Human','23657883','Dittwald_et_al_2013','Case-Set',null,'Here, we constructed bioinformatically a new genome-wide map of the DP-LCR-flanked regions in human genome build hg19 using a concept of LCR clusters. We then queried and cross referenced our database of 25,144 high-resolution genomic analyses performed on patients referred for Chromosomal Microarray Analysis (CMA). This approach enabled us to determine the relative frequencies in this clinical population of known recurrent genomic disorders, and also quantitate genomic architectural features genome-wide that are associated with individual locus events, to gain insights into the parameters rendering genomic instability.','Probe signal intensity',null,'Oligo aCGH','Not reported','GRCh37,GRCh37.p13,GRCh38,GRCh38.p2,NCBI36','GCA_000001405.14,GCA_000001405.15,GCA_000001405.17,GCF_000001405.12,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd8','9598,9606','Homo sapiens,Pan troglodytes','Chimpanzee,Human','18775914','Perry_et_al_2008b','Control Set',null,'In this study, we have used array-based comparative genomic hybridization (aCGH) on a human whole-genome tile-path (WGTP) platform comprised of 28,708 large-insert DNA clones to identify CNVs among the genomes of 30 unrelated chimpanzees (Pan troglodytes) and 30 unrelated humans from Africa.','Manual observation,Manual observation,Probe signal intensity,Probe signal intensity','CNVFinder,CNVFinder','FISH,PCR,BAC aCGH,BAC aCGH','Not reported,Not reported,Sanger H. sapiens Whole Genome Tile Path 28.7k v3,Sanger H. sapiens Whole Genome Tile Path 28.7k v3','GRCh37,GRCh37.p13,GRCh38,GRCh38.p2,NCBI36','GCA_000001405.14,GCA_000001405.15,GCA_000001405.17,GCF_000001405.12,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd80','9606','Homo sapiens','Human','23685542','Boone_et_al_2013','Case-Set',null,'A survey of deletions encompassing or disrupting known recessive disease genes in a clinical cohort of &gt;20,000 individuals.','Probe signal intensity,Probe signal intensity',null,'Oligo aCGH,Oligo aCGH','Not reported,Not reported','GRCh37,GRCh37.p13,GRCh38,GRCh38.p2,NCBI36','GCA_000001405.14,GCA_000001405.15,GCA_000001405.17,GCF_000001405.12,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd81','9606','Homo sapiens','Human','26730743','Low_et_al_2016','Case-Control',null,'Nasopharyngeal carcinoma (NPC), is a malignant tumour arising from the epithelial lining of the nasopharynx. While a number of studies on single nucleotide polymorphisms (SNPs) associated to susceptibility to NPC had previously been carried out, studies on CNV had not been prevalent even though the functional effects of these structural variants in pathogenesis can be significant. Hence, by genotyping more than 700,000 SNPs using IlluminaÂ® Human OmniExpress SNP-based microarrays, this study aims to discover associations between copy number variants (CNVs) in the genomes of our Malaysian Chinese study population with the susceptibility to NPC.','Probe signal intensity,SNP genotyping analysis','ABI QuantStudio 12K Flex Software v.1.0, Copy Caller v.2.0,Genome Studio v.3.1.6, cnvPartition v.3.1.6, PennCNV v.2011 Jun16), Partek (v.6.6)','qPCR,SNP array','ABI QuantStudio,Illumina Human OmniExpress','GRCh37,GRCh38,NCBI36','GCA_000001405.15,GCF_000001405.12,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd82','9593,9595,9597,9598,9600,9601,9606,37010,37011,37012,46359,406788,741158,756884','Gorilla beringei graueri,Gorilla gorilla,Gorilla gorilla diehli,Gorilla gorilla gorilla,Homo sapiens,Homo sapiens ssp. Denisova,Pan paniscus,Pan troglodytes,Pan troglodytes ellioti,Pan troglodytes schweinfurthii,Pan troglodytes troglodytes,Pan troglodytes verus,Pongo abelii,Pongo pygmaeus','Bonobo,Chimpanzee,Gorilla,Hominin,Human,Orangutan','23825009','Sudmant_et_al_2013','Control Set','http://eichlerlab.gs.washington.edu/greatape-cnv/','We assessed diversity and evolution of duplication and deletion diversity in the great ape lineage from 97 sequenced high-coverage great ape genomes.','Read depth,Read depth,Read depth,Read depth,Read depth,Probe signal intensity,Read depth,Read depth,Read depth,Read depth,Read depth',null,'Sequencing,Sequencing,Sequencing,Sequencing,Sequencing,Oligo aCGH,Sequencing,Sequencing,Sequencing,Sequencing,Sequencing','Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported','GRCh37,GRCh37.p13,GRCh38,GRCh38.p2,NCBI36,P_pygmaeus_2.0.2,Pan_troglodytes-2.1.3,Pan_troglodytes-2.1.4,gorGor3.1','GCA_000001405.14,GCA_000001405.15,GCA_000001405.17,GCA_000001515.3,GCA_000001515.4,GCA_000151905.1,GCF_000001405.12,GCF_000001405.13,GCF_000001405.28,GCF_000001545.3');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd83','9606','Homo sapiens','Human','24006408','Morak_et_al_2013','Case-Set',null,'The leukemia-initiating ETV6-RUNX1 (ER) fusion occurs already in utero but supporting genetic lesions are required to cause overt leukemia. To assess the timing of such B cell development- (PAX5, n=8; VpreB1, n=3), proliferation- and apoptosis-associated (BTG1, n=3; BMF, n=1) lesions, we analyzed Guthrie card DNA of 10 ER-positive leukemias for their presence. Despite our approach reaching a single cell detection level and the detection of genomic ER fusion in all Guthrie cards, we did not recover any of the specific secondary changes. These results are consistent with the prevailing notion that secondary aberrations occur in the postnatal period, but do not exclude the possibility that at least some of them emerge already prenatally and the affected clones simply have not yet reached a size that is detectable with the presently available approaches.','Sequence alignment',null,'Sequencing','Not reported','GRCh37,GRCh38','GCA_000001405.15,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd84','9606','Homo sapiens','Human','23893877','de_Ligt_et_al_2013','Case-Set',null,'An assessment of four published computational algorithms to identify clinically-relevant copy-number variation among patient-derived exome sequencing data. CNVs were concurrently assessed by two independent high resolution microarray platforms.','SNP genotyping analysis,Probe signal intensity,Read depth,Read depth,Probe signal intensity,Read depth,Read depth,SNP genotyping analysis',null,'SNP array,Oligo aCGH,Sequencing,Sequencing,Oligo aCGH,Sequencing,Sequencing,SNP array','Affymetrix 250k NspI array,NimbleGen, 4.2M probes,SOLiD 5500xl,SOLiD 5500xl,Affymetrix 2.7M microarray,SOLiD 5500xl,SOLiD 5500xl,Affymetrix CytoScanHD, 2.6M probes','GRCh37,GRCh38,GRCh38.p2','GCA_000001405.15,GCA_000001405.17,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd85','9606','Homo sapiens','Human','24038936','vanSilfhout_et_al_2013','Case-Control',null,'The study consists of a cohort of 5,531 consecutive patients with intellectual disability who were screened for CNVs using the Affymetrix 250K NspI single-nucleotide polymorphism (SNP) array platform, between 2006 and 2011.','SNP genotyping analysis',null,'SNP array','Affymetrix 250k NspI array','GRCh37,GRCh37.p13,GRCh38,GRCh38.p2,NCBI35','GCA_000001405.14,GCA_000001405.15,GCA_000001405.17,GCF_000001405.11,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd86','9606','Homo sapiens','Human','24094742','Poultney_et_al_2013','Case-Control',null,'Assessing burden of copy number variation in autism spectrum disorder using whole exome sequences','Read depth','XHMM (eXome Hidden Markov Model)','Sequencing','Illumina HiSeq 2000','GRCh37,GRCh38,GRCh38.p2','GCA_000001405.15,GCA_000001405.17,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd89','9606','Homo sapiens','Human','24478191','de_Boer_et_al_2014','Case-Set',null,'We report a novel LINE1-mediated insertion of a transcript from the TMF1 gene on chromosome 3 into the CYBB gene on the X chromosome in a Dutch male patient with chronic granulomatous disease. A 5.8-kb, incomplete and partly exonized TMF1 transcript was identified in intron 1 of CYBB in opposite orientation to the host gene. The sequence of the insertion showed the hallmarks of a retrotransposition event, with an antisense poly A tail, target site duplication and a consensus LINE1 endonuclease cleavage site. This insertion induced aberrant CYBB mRNA splicing, with inclusion of an extra 117-bp exon between exons 1 and 2 of CYBB. This extra exon contained a premature stop codon.','Sequence alignment',null,'Sequencing','ABI3130XL','GRCh37,GRCh38','GCA_000001405.15,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd9','37012','Pan troglodytes verus','Chimpanzee','16702545','Perry_et_al_2006','Control Set',null,'We used two array-based comparative genomic hybridization platforms to identify a total of 355 copy number variants (CNVs) in the genomes of 20 wild-born chimpanzees (Pan troglodytes).','Probe signal intensity,Probe signal intensity,Probe signal intensity',null,'BAC aCGH,qPCR,BAC aCGH','SpectralChip 2600,Not reported,GPL4010','GRCh37,GRCh37.p13,GRCh38,GRCh38.p2,NCBI34','GCA_000001405.14,GCA_000001405.15,GCA_000001405.17,GCF_000001405.10,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd90','9606','Homo sapiens','Human',null,'dbSNP_curated_variants','Collection',null,'Large indels previously submitted to dbSNP','Curated,Curated,Curated,Curated,Curated,Curated,Curated,Curated,Curated,Curated,Curated,Curated,Curated',null,'Curated,Curated,Curated,Curated,Curated,Curated,Curated,Curated,Curated,Curated,Curated,Curated,Curated','Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported,Not reported','GRCh37,GRCh38,GRCh38.p2','GCA_000001405.15,GCA_000001405.17,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd91','9606','Homo sapiens','Human','25293716','Sallustio_et_al_2014','Case-Control',null,'We performed a whole-genome screening of CNVs in IgAN patients, their healthy relatives and healthy subjects (HS). A total of 217 individuals consisting of 51 IgAN cases and 166 healthy relatives were included in the initial screening. We identified 148 IgAN-specific aberrations, 105 loss and 43 gain. Several CNVs overlapped with regions evidenced by previous genome-wide genetic studies. Moreover, we found that IgAN patients characterized by deteriorated renal function carried low copy numbers of a CNV in chromosome 3.','SNP genotyping analysis','Multiple Sample Analysis (MSA) algorithm to determine regions of concordant copy number aberrations across multiple samples. It is based on work published by Guttman, M., et al. in Assessing the Significance of Conserved Genomic Aberrations Using High Resolution Genomic Microarrays (PLoS Genetics Vol. 3, No. 8, e143 doi:10.1371/journal.pgen. 0030143 ).','SNP array','GPL13829','GRCh37,GRCh37.p13,GRCh38,NCBI36','GCA_000001405.14,GCA_000001405.15,GCF_000001405.12,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd92','9606','Homo sapiens','Human','24777449','Forsberg_et_al_2014','Control Set',null,'We describe age-related accumulation of copy number variation (CNVs) on periferal blood cells of elderly men from 2 Swedish cohorts of normally-aging subjects, [ULSAM|http://www2.pubcare.uu.se/ULSAM/] and [PIVUS|http://www.medsci.uu.se/pivus/]. DNA was tested on Illumina genotyping arrays and LogR Ratio and B-allele-frequency data were used to identify somatic copy number events on automosomes and sex chromosomes. Loss of chromosome Y was found in at least 8.2% and 20.5% of subjects respectively. Median survival time and all-cause mortality, as well as cancer mortality, were found to be significatively associated with loss of chromosome Y in the studied subjects.','SNP genotyping analysis,SNP genotyping analysis',null,'SNP array,SNP array','Illumina HumanOmni2.5-Quad BeadChip,HumanOmniExpress BeadChip','GRCh37,GRCh38,GRCh38.p2','GCA_000001405.15,GCA_000001405.17,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd93','9606','Homo sapiens','Human','24746959','Lindstrand_et_al_2014','Case-Set',null,'A custom high-resolution oligonucleotide array-comparative genomic hybridization scan of 772 genes prioritized from the ciliary proteome in patients with Bardet-Biedl Syndrome and controls','Sequence alignment,Probe signal intensity','UCSC genome browser Blat,Agilent Genomic Workbench ADM2','Sequencing,Oligo aCGH','Not reported,Agilent 180k','GRCh37,GRCh38','GCA_000001405.15,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd94','9606','Homo sapiens','Human','24823667','Helman_et_al_2014','Tumor vs. Matched-Normal',null,'Retrotransposons constitute a major source of genetic variation, and somatic retrotransposon insertions have been reported in cancer. Here, we applied TranspoSeq, a computational framework that identifies retrotransposon insertions from sequencing data, to whole-genomes from 200 tumor/normal pairs across 11 tumor types as part of The Cancer Genome Atlas (TCGA) Pan-Cancer Project. In addition to novel germline polymorphisms, we find 810 somatic retrotransposon insertions primarily in lung squamous, head and neck, colorectal and endometrial carcinomas. Many somatic retrotransposon insertions occur in known cancer genes. We find that high somatic retrotransposition rates in tumors are associated with high rates of genomic rearrangement and somatic mutation. Finally, we developed TranspoSeq-Exome to interrogate an additional 767 tumor samples with hybrid-capture exome data and discover 35 novel somatic retrotransposon insertions into exonic regions, including an insertion into an exon of the PTEN tumor suppressor gene. The results of this large-scale, comprehensive analysis of retrotransposon movement across tumor types suggest that somatic retrotransposon insertions may represent an important class of structural variation in cancer.','Split read mapping,Manual observation,Split read and paired-end mapping',null,'Sequencing,PCR,Sequencing','Not reported,Not reported,Not reported','GRCh37,GRCh38,GRCh38.p2','GCA_000001405.15,GCA_000001405.17,GCF_000001405.13,GCF_000001405.28');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd95','10092','Mus musculus domesticus','Mouse','26149421','Pezer_et_al_2015','Control Set',null,'We applied a read depth approach on next-generation sequencing data to study CNVs in 27 wild mice belonging to one ancestral and three derived populations.','Read depth','CNVnator','Sequencing','Illumina HiSeq2000','GRCm38,GRCm38.p3,MGSCv37','GCF_000001635.16,GCF_000001635.20,GCF_000001635.23');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd96','4527','Oryza','Rice','27025496','Bai_et_al_2016','Control Set',null,'Copy number variation (CNV), a complex genomic rearrangement, has been extensively studied in human and other species. Studies of CNV in plants have lagged behind, however. We generated the CNV map comprising 9,196 deletions relative to the reference in Oryza species derived from the published next-generation sequencing data. &gt;80% of them were classified as insertions using the outgroup Oryza glaberrima. Coding region of 1,675 annotated genes was affected by CNVs with the function enriched in defense response pathway. 28 functional genes affected by CNV were validated, including OsMADS56, BPH14, OsDCL2b and OsMADS30, suggesting the potential role of CNV on phenotypes. 41% conserved CNV genes are non-collinear compared to O. glaberrima. Ã¢Â€ÂœCopy and pasteÃ¢Â€Â of genomic fragments accompany with transposon and double streak break (DSB) repair processes was one origin of them. Comprehensive analysis of mutational mechanisms that generate CNV suggested non-homologous end-joining (NHEJ) and mobile element insertion (MEI) as the major determinants cause rearrangement. &gt;80% CNVs are shared between species indicating that most of them are from the same gene pool, and predate the rice domestication or improvement. Our CNV map provides an entry point for future research on the role of CNV in Oryza genome evolution and adaptation by sequencing-based association studies.','Merging,Paired-end mapping,Split read mapping,Read depth','CNVnator,BreakDancer,Pindel','Merging,Sequencing,Sequencing,Sequencing','See merged experiments,Illumina GA2,Illumina GA2,Illumina GA2','IRGSP-1.0','GCF_001433935.1');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd97','9606','Homo sapiens','Human','25026903','Lou_et_al_2014','Control Set',null,'This study consists of three admixed populations (Kazakh, Kirgiz and Uyghur) in Xinjiang province, northwest of China. We used Affymetrix Genome-Wide Human SNP Array 6.0 to detect the copy number variations in these populations. The samples were all normal individuals which can be used as control set for future study.','SNP genotyping analysis','Birdsuite v1.5.2','SNP array','Not reported','GRCh37,GRCh38,GRCh38.p2','GCA_000001405.15,GCA_000001405.17,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd98','9606','Homo sapiens','Human','25246103','Campbell_et_al_2014b','Case-Set',null,'Breakpoint determination of HERV-HERV mediated CNVs in individuals with diverse phenotypes from a clinical laboratory.','Sequence alignment',null,'Sequencing','Not reported','GRCh37,GRCh38','GCA_000001405.15,GCF_000001405.13');
Insert into dgva_study_browser (STUDY_ACCESSION,TAXONOMY_IDS,SCIENTIFIC_NAMES,COMMON_NAMES,PUBMED_IDS,DISPLAY_NAME,STUDY_TYPE,STUDY_URL,STUDY_DESCRIPTION,ANALYSIS_TYPES,DETECTION_METHODS,METHOD_TYPES,PLATFORM_NAMES,ASSEMBLY_NAMES,ASSEMBLY_ACCESSIONS) values ('nstd99','9606','Homo sapiens','Human','25765185','Thareja_et_al_2015','Control Set',null,'Comprehensive analysis of a personal genome of Persian ancestry from Kuwait','Read depth and paired-end mapping',null,'Sequencing','Illumina HiSeq 2000','GRCh37,GRCh38,GRCh38.p2','GCA_000001405.15,GCA_000001405.17,GCF_000001405.13,GCF_000001405.28');